# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.3 (2017-03-06) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |(EN)                         |
|collate  |en_US.UTF-8                  |
|tz       |Zulu                         |
|date     |2017-04-05                   |

## Packages

|package   |*  |version    |date       |source                           |
|:---------|:--|:----------|:----------|:--------------------------------|
|covr      |   |2.2.2      |2017-01-05 |cran (@2.2.2)                    |
|DBI       |   |0.6-1      |2017-04-03 |Github (rstats-db/DBI@c579130)   |
|hms       |   |0.3        |2016-11-22 |cran (@0.3)                      |
|knitr     |   |1.15.1     |2016-11-22 |cran (@1.15.1)                   |
|magrittr  |   |1.5        |2014-11-22 |CRAN (R 3.3.1)                   |
|rmarkdown |   |1.4        |2017-03-24 |cran (@1.4)                      |
|rprojroot |   |1.2        |2017-01-16 |cran (@1.2)                      |
|RSQLite   |   |1.1-2      |2017-01-08 |cran (@1.1-2)                    |
|testthat  |   |1.0.2.9000 |2017-02-27 |Github (hadley/testthat@b72a228) |
|xml2      |   |1.1.1      |2017-01-24 |cran (@1.1.1)                    |

# Check results

143 packages

|package            |version   | errors| warnings| notes|
|:------------------|:---------|------:|--------:|-----:|
|acc                |1.3.3     |      0|        0|     0|
|afmToolkit         |0.0.1     |      0|        0|     0|
|anamiR             |1.0.1     |      0|        0|     1|
|annaffy            |1.46.0    |      0|        0|     2|
|annmap             |1.16.0    |      0|        2|     5|
|annotate           |1.52.1    |      0|        0|     1|
|AnnotationDbi      |1.36.2    |      0|        1|     5|
|AnnotationForge    |1.16.1    |      0|        0|     1|
|AnnotationFuncs    |1.24.0    |      0|        0|     3|
|AnnotationHubData  |1.4.1     |      0|        0|     3|
|archivist          |2.1.2     |      0|        0|     2|
|aroma.affymetrix   |3.1.0     |      0|        0|     0|
|BatchExperiments   |1.4.1     |      0|        0|     2|
|BatchJobs          |1.6       |      0|        0|     0|
|bibliospec         |0.0.4     |      0|        0|     0|
|BIEN               |1.1.0     |      0|        0|     0|
|biglm              |0.9-1     |      0|        0|     5|
|bigrquery          |0.3.0     |      0|        0|     0|
|bioassayR          |1.12.1    |      0|        0|     1|
|ChemmineR          |2.26.1    |      1|        0|     0|
|ChIPpeakAnno       |3.8.9     |      0|        1|     2|
|chunked            |0.3       |      0|        0|     0|
|CITAN              |2015.12-2 |      0|        0|     0|
|cn.farms           |1.22.0    |      1|        0|     1|
|convey             |0.1.0     |      1|        0|     0|
|datamap            |0.1-1     |      0|        0|     2|
|dbfaker            |0.1.0     |      0|        0|     0|
|DBItest            |1.4       |      0|        0|     0|
|DECIPHER           |2.2.0     |      0|        0|     3|
|DMwR2              |0.0.2     |      0|        0|     0|
|dplyr              |0.5.0     |      0|        0|     1|
|eiR                |1.14.1    |      1|        2|     6|
|eisa               |1.26.0    |      0|        0|     2|
|emuR               |0.2.2     |      0|        0|     0|
|ensembldb          |1.6.2     |      0|        0|     2|
|etl                |0.3.5     |      0|        0|     0|
|ETLUtils           |1.3       |      0|        0|     0|
|ExperimentHubData  |1.0.0     |      0|        0|     1|
|filehashSQLite     |0.2-4     |      0|        0|     3|
|frma               |1.26.0    |      0|        0|     2|
|frmaTools          |1.26.0    |      0|        0|     4|
|gcbd               |0.2.6     |      0|        0|     0|
|GenomicFeatures    |1.26.4    |      0|        0|     2|
|genomicper         |1.6       |      0|        0|     0|
|Genominator        |1.28.0    |      0|        0|     4|
|GenVisR            |1.4.1     |      0|        0|     0|
|gmDatabase         |0.5.0     |      0|        0|     0|
|GOFunction         |1.22.0    |      0|        1|     5|
|GWASTools          |1.20.0    |      0|        0|     1|
|implyr             |0.1.0     |      0|        0|     0|
|imputeMulti        |0.6.4     |      0|        0|     1|
|isobar             |1.20.0    |      0|        0|     5|
|ITALICS            |2.34.0    |      0|        0|     5|
|keggorthology      |2.26.0    |      0|        0|     3|
|knitr              |1.15.1    |      0|        0|     0|
|liteq              |1.0.0     |      0|        0|     0|
|lumi               |2.26.4    |      0|        1|     3|
|macleish           |0.3.0     |      0|        0|     0|
|marmap             |0.9.6     |      0|        0|     0|
|mdgsa              |1.6.0     |      0|        0|     2|
|mdsr               |0.1.3     |      0|        0|     1|
|MetaIntegrator     |1.0.3     |      0|        0|     0|
|mgsa               |1.22.0    |      0|        1|     4|
|miRNAmeConverter   |1.2.0     |      0|        0|     1|
|miRNAtap           |1.8.0     |      0|        0|     2|
|mitools            |2.3       |      0|        0|     1|
|MmPalateMiRNA      |1.24.0    |      0|        0|     4|
|MonetDBLite        |0.3.1     |      0|        0|     1|
|MonetDB.R          |1.0.1     |      0|        0|     0|
|oai                |0.2.2     |      0|        0|     0|
|oce                |0.9-21    |      1|        0|     1|
|odbc               |1.0.1     |      1|        0|     0|
|ODB                |1.1.1     |      0|        0|     2|
|oligo              |1.38.0    |      1|        0|     8|
|ora                |2.0-1     |      1|        0|     0|
|PAnnBuilder        |1.38.0    |      0|        3|     1|
|pdInfoBuilder      |1.38.0    |      0|        0|     1|
|pitchRx            |1.8.2     |      0|        0|     1|
|PivotalR           |0.1.18.3  |      0|        0|     0|
|plethy             |1.12.1    |      2|        0|     3|
|poplite            |0.99.17.3 |      0|        0|     0|
|postGIStools       |0.2.1     |      0|        0|     0|
|ProjectTemplate    |0.7       |      0|        0|     0|
|quantmod           |0.4-7     |      0|        0|     1|
|RecordLinkage      |0.4-10    |      0|        0|     0|
|redcapAPI          |1.3       |      0|        0|     1|
|refGenome          |1.7.3     |      0|        0|     0|
|RImmPort           |1.2.0     |      0|        1|     1|
|RJDBC              |0.2-5     |      0|        1|     1|
|RmiR               |1.30.0    |      0|        0|     5|
|RMySQL             |0.10.11   |      0|        0|     0|
|rnaSeqMap          |2.32.0    |      0|        1|     4|
|RObsDat            |16.03     |      0|        0|     0|
|RODBCDBI           |0.1.1     |      0|        0|     0|
|Roleswitch         |1.12.0    |      0|        0|     2|
|ROracle            |1.3-1     |      1|        0|     0|
|rpostgisLT         |0.4.0     |      0|        0|     0|
|rpostgis           |1.2.0     |      0|        0|     0|
|RPostgreSQL        |0.4-1     |      0|        0|     3|
|RPresto            |1.2.1     |      0|        0|     0|
|RQDA               |0.2-8     |      1|        0|     1|
|RRedshiftSQL       |0.1.2     |      0|        0|     0|
|RSQLite            |1.1-2     |      1|        0|     1|
|rTRM               |1.12.0    |      0|        0|     1|
|SEERaBomb          |2016.2    |      0|        0|     0|
|sejmRP             |1.3.4     |      0|        0|     0|
|seqplots           |1.12.0    |      1|        0|     3|
|sf                 |0.4-1     |      0|        0|     1|
|smnet              |2.1.1     |      0|        0|     0|
|snplist            |0.17      |      0|        0|     0|
|sparklyr           |0.5.3     |      0|        0|     0|
|sparkwarc          |0.1.1     |      0|        0|     0|
|specL              |1.8.0     |      0|        1|     3|
|sqldf              |0.4-10    |      0|        1|     2|
|sqliter            |0.1.0     |      0|        0|     0|
|sqlutils           |1.2       |      0|        0|     3|
|storr              |1.0.1     |      0|        0|     0|
|stream             |1.2-4     |      0|        0|     1|
|survey             |3.31-5    |      0|        0|     0|
|tcpl               |1.2.2     |      0|        0|     1|
|tigreBrowserWriter |0.1.4     |      0|        0|     0|
|tigre              |1.28.1    |      0|        0|     2|
|timeseriesdb       |0.2.1     |      0|        0|     1|
|topGO              |2.26.0    |      0|        0|     4|
|toxboot            |0.1.1     |      0|        0|     1|
|TScompare          |2015.4-1  |      0|        0|     1|
|TSdata             |2016.8-1  |      0|        1|     0|
|TSdbi              |2015.7-1  |      0|        0|     0|
|TSfame             |2015.4-1  |      0|        0|     1|
|TSmisc             |2016.8-1  |      0|        0|     0|
|TSMySQL            |2015.4-1  |      0|        0|     0|
|TSodbc             |2015.4-1  |      0|        0|     0|
|TSPostgreSQL       |2015.4-1  |      0|        0|     0|
|TSsdmx             |2016.8-1  |      0|        0|     0|
|TSSQLite           |2015.4-1  |      0|        0|     0|
|TSsql              |2015.1-2  |      0|        0|     1|
|twitteR            |1.1.9     |      0|        0|     0|
|Uniquorn           |1.2.0     |      0|        0|     2|
|UPMASK             |1.1       |      0|        0|     0|
|VariantAnnotation  |1.20.3    |      0|        1|     3|
|VariantFiltering   |1.10.1    |      0|        1|     4|
|vmsbase            |2.1.3     |      1|        0|     0|
|WhopGenome         |0.9.7     |      0|        0|     0|

## acc (1.3.3)
Maintainer: Jaejoon Song <jjsong2@mdanderson.org>

0 errors | 0 warnings | 0 notes

## afmToolkit (0.0.1)
Maintainer: Rafael Benitez <rabesua@uv.es>

0 errors | 0 warnings | 0 notes

## anamiR (1.0.1)
Maintainer: Ti-Tai Wang <r04945002@ntu.edu.tw>  
Bug reports: https://github.com/AllenTiTaiWang/anamiR/issues

0 errors | 0 warnings | 1 note 

```
checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘SummarizedExperiment’
A package should be listed in only one of these fields.
```

## annaffy (1.46.0)
Maintainer: Colin A. Smith <colin@colinsmith.org>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘methods’ which was already attached by Depends.
  Please remove these calls from your code.
'library' or 'require' call to ‘tcltk’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘GO.db’ ‘KEGG.db’ ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Missing object imported by a ':::' call: ‘Biobase:::biocReposList’

checking R code for possible problems ... NOTE
File ‘annaffy/R/zzz.R’:
  .onLoad calls:
    require(methods)

Package startup functions should not change the search path.
See section ‘Good practice’ in '?.onAttach'.

.aaf.goterm: no visible binding for global variable ‘GOTERM’
aafGO: no visible global function definition for ‘GO_dbfile’
... 47 lines ...
Undefined global functions or variables:
  GOTERM GO_dbfile KEGGPATHID2NAME available.packages browseURL
  contrib.url exprs featureNames install.packages sampleNames tclvalue
  text tkbind tkbutton tkcurselection tkdelete tkdestroy tkframe tkget
  tkgrid tkinsert tklabel tklistbox tkscrollbar tkselection.clear tkset
  tktoplevel tkwait.window tkwm.resizable tkwm.title tkyview userQuery
Consider adding
  importFrom("graphics", "text")
  importFrom("utils", "available.packages", "browseURL", "contrib.url",
             "install.packages")
to your NAMESPACE file.
```

## annmap (1.16.0)
Maintainer: Chris Wirth <Christopher.Wirth@cruk.manchester.ac.uk>

0 errors | 2 warnings | 5 notes

```
checking Rd cross-references ... WARNING
Missing link or links in documentation object 'plot.ngs.Rd':
  ‘[IRanges]{Rle}’

See section 'Cross-references' in the 'Writing R Extensions' manual.


checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
    auto-pst-pdf will not work!
.

See the auto-pst-pdf package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
... 8 lines ...
l.134         Or turn off auto-pst-pdf.}
                                        %
/usr/share/texlive/texmf-dist/tex/latex/auto-pst-pdf/auto-pst-pdf.sty:134:  ==>
 Fatal error occurred, no output PDF file produced!
Transcript written on annmap.log.
/usr/bin/texi2dvi: pdflatex exited with bad status, quitting.
Makefile:7: recipe for target 'pdf' failed
make: *** [pdf] Error 1
Error in buildVignettes(dir = "/home/muelleki/git/R/DBI/revdep/checks/annmap.Rcheck/vign_test/annmap") : 
  running 'make' failed
Execution halted

checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘rjson’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘.make.hash’

checking R code for possible problems ... NOTE
.coerce: no visible global function definition for ‘setNames’
.coerce: no visible binding for global variable ‘v’
.coerce: no visible binding for global variable ‘k’
.get.stats.cache: no visible global function definition for
  ‘flush.console’
.layout.genes: no visible global function definition for ‘strwidth’
.load.and.parse: no visible global function definition for ‘fromJSON’
.read.databases: no visible global function definition for ‘read.table’
.single.transcript.coords.to.genome: no visible global function
... 25 lines ...
Undefined global functions or variables:
  GeneRegionTrack Rle flush.console fromJSON install.packages k menu
  modifyList rainbow read.table rgb runValue seqlevels seqlevels<-
  setNames strwidth tail v write.table
Consider adding
  importFrom("grDevices", "rainbow", "rgb")
  importFrom("graphics", "strwidth")
  importFrom("stats", "setNames")
  importFrom("utils", "flush.console", "install.packages", "menu",
             "modifyList", "read.table", "tail", "write.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'annmap.coords.Rd':
  \usage lines wider than 90 characters:
       transcriptCoordsToGenome( transcript.ids, position=1, as.vector=FALSE, check.bounds=TRUE, truncate=TRUE, cds=FALSE )
       genomeToTranscriptCoords( position, transcript.ids, as.vector=FALSE, check.bounds=TRUE, end=c( 'none', 'both', '5', '3' ) )
       proteinCoordsToGenome( protein.ids, position=1, as.vector=FALSE, check.bounds=TRUE, truncate=TRUE )

Rd file 'annmap.range.Rd':
  \examples lines wider than 100 characters:
         df = data.frame( chr=c( '17', 'X' ), start=c( 7510000, 1000000 ), end=c( 7550000, 1500000 ), strand=c( -1, -1 ) )
... 24 lines ...
       genomicExonDepthPlot( .exons, start, end, exon.depth.alpha=0.1, exon.depth.col='black', ... )

Rd file 'plot.ngs.Rd':
  \usage lines wider than 90 characters:
       convertBamToRle( bam.file.name, chr, start, end, chr.name.mapping=function( name ){ name } )
       ngsTracePlotter( rle.data, start, end, ylim, trace.label.properties=list(), smoothing.function=function( rle, ... ) { IRanges::runmea ... [TRUNCATED]
                          trace.clip='inherit', trace.draw.scale=FALSE, trace.bor='transparent', trace.pad=c(0,0), ... )
       ngsBridgePlot( xrange, data=list(), main=NULL, sub=NULL, highlights=NULL, trace.plotter=ngsTracePlotter, genome.layout.weight=4,
                    trace.scale=ngsTraceScale, trace.draw.scale=NULL, trace.match.strand=TRUE, probe.plot=NULL, exon.depth.plot=genomicExonD ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## annotate (1.52.1)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 0 warnings | 1 note 

```
checking Rd line widths ... NOTE
Rd file 'readGEOAnn.Rd':
  \usage lines wider than 90 characters:
                            "https://www.ncbi.nlm.nih.gov/geo/query/browse.cgi?view=platforms&prtype=SAGE&dtype=SAGE")

These lines will be truncated in the PDF manual.
```

## AnnotationDbi (1.36.2)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 1 warning  | 5 notes

```
checking for unstated dependencies in ‘tests’ ... WARNING
'library' or 'require' call not declared from: ‘org.testing.db’

checking installed package size ... NOTE
  installed size is  8.6Mb
  sub-directories of 1Mb or more:
    extdata   6.0Mb

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’ ‘utils’ ‘stats4’ ‘BiocGenerics’ ‘Biobase’ ‘IRanges’ ‘DBI’ ‘RSQLite’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘GO.db’ ‘KEGG.db’ ‘RSQLite’ ‘graph’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
.selectInp8: no visible global function definition for ‘.resort’
annotMessage: no visible binding for global variable ‘pkgName’
createORGANISMSeeds: no visible global function definition for
  ‘makeAnnDbMapSeeds’
makeGOGraph: no visible binding for global variable ‘GOBPPARENTS’
makeGOGraph: no visible binding for global variable ‘GOMFPARENTS’
makeGOGraph: no visible binding for global variable ‘GOCCPARENTS’
makeGOGraph: no visible global function definition for ‘ftM2graphNEL’
Undefined global functions or variables:
  .resort GOBPPARENTS GOCCPARENTS GOMFPARENTS ftM2graphNEL
  makeAnnDbMapSeeds pkgName

checking Rd line widths ... NOTE
Rd file 'inpIDMapper.Rd':
  \examples lines wider than 100 characters:
       YeastUPSingles = inpIDMapper(ids, "HOMSA", "SACCE", destIDType="UNIPROT", keepMultDestIDMatches = FALSE)

These lines will be truncated in the PDF manual.
```

## AnnotationForge (1.16.1)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.0Mb
  sub-directories of 1Mb or more:
    AnnDbPkg-templates   1.7Mb
    extdata              3.3Mb
```

## AnnotationFuncs (1.24.0)
Maintainer: Stefan McKinnon Edwards <stefan.hoj-edwards@agrsci.dk>  
Bug reports: 

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘AnnotationDbi’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
getOrthologs: no visible global function definition for ‘dbmeta’
getOrthologs: no visible global function definition for ‘dbconn’
getOrthologs: no visible global function definition for ‘dbfile’
getOrthologs: no visible global function definition for ‘direction’
translate: no visible global function definition for ‘dbmeta’
translate: no visible global function definition for ‘dbconn’
translate: no visible global function definition for ‘stack’
Undefined global functions or variables:
  dbconn dbfile dbmeta direction stack
Consider adding
  importFrom("utils", "stack")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'getOrthologs.Rd':
  \examples lines wider than 100 characters:
     bovine.ensembl <- getOrthologs(c(1,2,3), hom.Hs.inpBOSTA, 'BOSTA', pre.from=org.Hs.egENSEMBLPROT, post.from=org.Bt.egENSEMBLPROT2EG)
     hs2bt.refseqs2 <- lapply(bovine.ensembl, translate, from=org.Bt.egREFSEQ, simplify=TRUE) # simplify=TRUE is very important here!

These lines will be truncated in the PDF manual.
```

## AnnotationHubData (1.4.1)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 0 warnings | 3 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘appveyor.yml’

checking dependencies in R code ... NOTE
Missing object imported by a ':::' call: ‘AnnotationHub:::.db_connection’
Unexported object imported by a ':::' call: ‘OrganismDbi:::.packageTaxIds’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.NCBIMetadataFromUrl: no visible binding for global variable ‘results’
.NCBIMetadataFromUrl: no visible binding for global variable ‘specData’
.makeComplexGR: no visible binding for global variable ‘seqname’
jsonPath: no visible binding for global variable ‘SourceFile’
jsonPath: no visible binding for global variable ‘HubRoot’
makeAnnotationHubMetadata : <anonymous>: no visible binding for global
  variable ‘Title’
makeAnnotationHubMetadata : <anonymous>: no visible binding for global
  variable ‘Description’
... 52 lines ...
  SourceFile SourceType SourceUrl SourceVersion Species TaxonomyId
  Title ahroot bucket checkTrue read.csv results seqname specData
  suppresWarnings
Consider adding
  importFrom("utils", "read.csv")
to your NAMESPACE file.

Found the following calls to data() loading into the global environment:
File ‘AnnotationHubData/R/makeNCBIToOrgDbs.R’:
  data(specData, package = "GenomeInfoDb")
See section ‘Good practice’ in ‘?data’.
```

## archivist (2.1.2)
Maintainer: Przemyslaw Biecek <przemyslaw.biecek@gmail.com>  
Bug reports: https://github.com/pbiecek/archivist/issues

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘archivist.github’

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘archivist.github’
```

## aroma.affymetrix (3.1.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.affymetrix/issues

0 errors | 0 warnings | 0 notes

## BatchExperiments (1.4.1)
Maintainer: Michel Lang <michellang@gmail.com>  
Bug reports: https://github.com/tudo-r/BatchExperiments/issues

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘BatchJobs:::addIntModulo’ ‘BatchJobs:::buffer’
  ‘BatchJobs:::checkDir’ ‘BatchJobs:::checkId’ ‘BatchJobs:::checkIds’
  ‘BatchJobs:::checkPart’ ‘BatchJobs:::createShardedDirs’
  ‘BatchJobs:::dbConnectToJobsDB’ ‘BatchJobs:::dbCreateJobStatusTable’
  ‘BatchJobs:::dbDoQuery’ ‘BatchJobs:::dbFindDone’
  ‘BatchJobs:::dbFindRunning’ ‘BatchJobs:::dbRemoveJobs’
  ‘BatchJobs:::dbSelectWithIds’ ‘BatchJobs:::getJobDirs’
  ‘BatchJobs:::getJobInfoInternal’ ‘BatchJobs:::getKillJob’
  ‘BatchJobs:::getListJobs’ ‘BatchJobs:::getRandomSeed’
  ‘BatchJobs:::getResult’ ‘BatchJobs:::isRegistryDir’
  ‘BatchJobs:::makeRegistryInternal’ ‘BatchJobs:::saveRegistry’
  ‘BatchJobs:::seeder’ ‘BatchJobs:::syncRegistry’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
addExperiments.ExperimentRegistry: no visible global function
  definition for ‘is’
applyJobFunction.ExperimentRegistry: no visible global function
  definition for ‘setNames’
calcDynamic: no visible global function definition for ‘setNames’
checkExperimentRegistry: no visible global function definition for
  ‘head’
dbSummarizeExperiments: no visible global function definition for
  ‘setNames’
designIterator: no visible global function definition for ‘setNames’
getIndex : exprToIndex: no visible global function definition for
  ‘capture.output’
getProblemFilePaths: no visible global function definition for
  ‘setNames’
updateRegistry.ExperimentRegistry: no visible global function
  definition for ‘packageVersion’
Undefined global functions or variables:
  capture.output head is packageVersion setNames
Consider adding
  importFrom("methods", "is")
  importFrom("stats", "setNames")
  importFrom("utils", "capture.output", "head", "packageVersion")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## BatchJobs (1.6)
Maintainer: Bernd Bischl <bernd_bischl@gmx.net>  
Bug reports: https://github.com/tudo-r/BatchJobs/issues

0 errors | 0 warnings | 0 notes

## bibliospec (0.0.4)
Maintainer: Witold E. Wolski <wew@fgcz.ethz.ch>  
Bug reports: https://github.com/protViz/bibliospec/issues

0 errors | 0 warnings | 0 notes

## BIEN (1.1.0)
Maintainer: Brian Maitner <bmaitner@gmail.com>

0 errors | 0 warnings | 0 notes

## biglm (0.9-1)
Maintainer: Thomas Lumley <tlumley@u.washington.edu>

0 errors | 0 warnings | 5 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘test’

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘DBI’ ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking foreign function calls ... NOTE
Call with DUP:
   .Fortran("regcf", as.integer(p), as.integer(p * p/2), bigQR$D, 
       bigQR$rbar, bigQR$thetab, bigQR$tol, beta = numeric(p), nreq = as.integer(nvar), 
       ier = integer(1), DUP = FALSE)
DUP is no longer supported and will be ignored.

checking R code for possible problems ... NOTE
bigglm.RODBC: no visible global function definition for ‘odbcQuery’
bigglm.RODBC : chunk: no visible global function definition for
  ‘odbcQuery’
bigglm.RODBC : chunk: no visible global function definition for
  ‘sqlGetResults’
bigglm,ANY-DBIConnection: no visible global function definition for
  ‘dbSendQuery’
bigglm,ANY-DBIConnection: no visible global function definition for
  ‘dbClearResult’
bigglm,ANY-DBIConnection : chunk: no visible global function definition
  for ‘dbClearResult’
bigglm,ANY-DBIConnection : chunk: no visible global function definition
  for ‘dbSendQuery’
bigglm,ANY-DBIConnection : chunk: no visible global function definition
  for ‘fetch’
Undefined global functions or variables:
  dbClearResult dbSendQuery fetch odbcQuery sqlGetResults

Found the following calls to data() loading into the global environment:
File ‘biglm/R/bigglm.R’:
  data(reset = TRUE)
  data(reset = FALSE)
See section ‘Good practice’ in ‘?data’.
```

## bigrquery (0.3.0)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/rstats-db/bigrquery/issues

0 errors | 0 warnings | 0 notes

## bioassayR (1.12.1)
Maintainer: Tyler Backman <tbackman@lbl.gov>  
Bug reports: https://github.com/TylerBackman/bioassayR/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
crossReactivityPrior: no visible global function definition for ‘sd’
crossReactivityProbability : <anonymous>: no visible global function
  definition for ‘pbeta’
Undefined global functions or variables:
  pbeta sd
Consider adding
  importFrom("stats", "pbeta", "sd")
to your NAMESPACE file.
```

## ChemmineR (2.26.1)
Maintainer: Thomas Girke <thomas.girke@ucr.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ChemmineR’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/DBI/revdep/checks/ChemmineR.Rcheck/00install.out’ for details.
```

## ChIPpeakAnno (3.8.9)
Maintainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>,
 Jianhong Ou <Jianhong.ou@umassmed.edu>

0 errors | 1 warning  | 2 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-25 (ChIPpeakAnno.Rmd) 
Error: processing vignette 'ChIPpeakAnno.Rmd' failed with diagnostics:
could not find function "doc_date"
Execution halted


checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘grid’ ‘IRanges’ ‘Biostrings’ ‘GenomicRanges’ ‘S4Vectors’
  ‘VennDiagram’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 20.8Mb
  sub-directories of 1Mb or more:
    R         1.6Mb
    data     12.7Mb
    extdata   5.2Mb
```

## chunked (0.3)
Maintainer: Edwin de Jonge <edwindjonge@gmail.com>  
Bug reports: https://github.com/edwindj/chunked/issues

0 errors | 0 warnings | 0 notes

## CITAN (2015.12-2)
Maintainer: Marek Gagolewski <gagolews@ibspan.waw.pl>  
Bug reports: https://github.com/Rexamine/CITAN/issues

0 errors | 0 warnings | 0 notes

## cn.farms (1.22.0)
Maintainer: Andreas Mitterecker <mitterecker@bioinf.jku.at>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘cn.farms-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotEvalIc
> ### Title: Creates a plot with known regions and a numeric vector
> ### Aliases: plotEvalIc
> 
> ### ** Examples
> 
> load(system.file("exampleData/slData.RData", package = "cn.farms"))
> load(system.file("exampleData/testSegments.RData", package = "cn.farms"))
> plotEvalIc(slData, fData(testSegments),
+      variable = assayData(slData)$L_z[, 1], 23)
Error in loadNamespace(name) : there is no package called ‘KernSmooth’
Calls: plotEvalIc ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted

checking R code for possible problems ... NOTE
calcDistance: no visible global function definition for ‘density’
callSummarizeH01: no visible binding for global variable ‘var’
callSummarizeH01: no visible global function definition for ‘rnorm’
createAnnotation: no visible global function definition for
  ‘installed.packages’
determineBaselineArray: no visible binding for global variable ‘median’
flcSnp6Std: no visible global function definition for ‘lowess’
flcSnp6Std: no visible global function definition for ‘approx’
flcSnp6StdH01: no visible global function definition for ‘lowess’
... 102 lines ...
             "order.dendrogram", "pchisq", "rnorm", "runif", "sd", "var")
  importFrom("utils", "assignInNamespace", "getFromNamespace", "head",
             "installed.packages", "packageDescription")
to your NAMESPACE file.

Found the following assignments to the global environment:
File ‘cn.farms/R/sFclusterFunctions.R’:
  assign("var1", 99, pos = globalenv())
  assign("var2", 101, pos = globalenv())
File ‘cn.farms/R/sFsnowfall-internal.R’:
  assign(name, value, envir = globalenv())
```

## convey (0.1.0)
Maintainer: Djalma Pessoa <pessoad@gmail.com>  
Bug reports: https://github.com/djalmapessoa/convey/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [101s/105s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  [9] 2.45 - 2.22 ==  0.22891
  
  
  testthat results ================================================================
  OK: 259 SKIPPED: 0 FAILED: 7
  1. Error: (unknown) (@test-svyarpr.R#18) 
  2. Error: (unknown) (@test-svyarpt.R#14) 
  3. Error: (unknown) (@test-svypoormed.R#14) 
  4. Failure: compare results convey vs vardpoor (@test-svyqsr.R#45) 
  5. Error: (unknown) (@test-svyrmir.R#31) 
  6. Failure: compare results convey vs vardpoor (@test-svyrmpg.R#48) 
  7. Failure: compare results convey vs vardpoor (@test-svyrmpg.R#50) 
  
  Error: testthat unit tests failed
  Execution halted
```

## datamap (0.1-1)
Maintainer: Jeffrey Horner <jeffrey.horner@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘DBI’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
.onLoad : <anonymous>: no visible global function definition for
  ‘read.table’
.onLoad : <anonymous>: no visible global function definition for
  ‘dbConnect’
.onLoad : <anonymous>: no visible global function definition for
  ‘dbListTables’
.onLoad : <anonymous>: no visible global function definition for
  ‘dbExistsTable’
.onLoad : <anonymous>: no visible global function definition for
... 9 lines ...
  dbConnect dbDisconnect dbExistsTable dbListTables dbReadTable
  dbRemoveTable dbWriteTable read.table
Consider adding
  importFrom("utils", "read.table")
to your NAMESPACE file.

Found the following calls to attach():
File ‘datamap/R/datamap.R’:
  attach(.Call("CreateUserDB", map, PACKAGE = "datamap"), pos, 
    name, warn.conflicts)
See section ‘Good practice’ in ‘?attach’.
```

## dbfaker (0.1.0)
Maintainer: Hari Mohanraj <hari@shipt.com>

0 errors | 0 warnings | 0 notes

## DBItest (1.4)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/rstats-db/DBItest/issues

0 errors | 0 warnings | 0 notes

## DECIPHER (2.2.0)
Maintainer: Erik Wright <DECIPHER@cae.wisc.edu>

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is  9.1Mb
  sub-directories of 1Mb or more:
    data      2.5Mb
    doc       3.8Mb
    extdata   1.4Mb

checking foreign function calls ... NOTE
Registration problems:
  symbol ‘functionCall’ in the local frame:
   .Call(functionCall, myXStringSet, as.numeric(subMatrix), gapOpening, 
       gapExtension, gapLetter, shiftPenalty, threshold, weight, 
       PACKAGE = "DECIPHER")
  symbol ‘consensusProfile’ in the local frame:
   .Call(consensusProfile, pattern, p.weight, NULL, PACKAGE = "DECIPHER")
  symbol ‘consensusProfile’ in the local frame:
   .Call(consensusProfile, subject, s.weight, NULL, PACKAGE = "DECIPHER")
... 11 lines ...
       structureMatrix)
  Evaluating ‘compression[1]’ during check gives error
‘object 'compression' not found’:
   .Call(compression[1], x, 2L - length(compression), compressRepeats, 
       processors, PACKAGE = "DECIPHER")
  Evaluating ‘compression[1]’ during check gives error
‘object 'compression' not found’:
   .Call(compression[1], x, 2L - length(compression), processors, 
       PACKAGE = "DECIPHER")
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.

checking R code for possible problems ... NOTE
DesignSignatures: no visible binding for global variable ‘deltaHrules’
Undefined global functions or variables:
  deltaHrules
```

## DMwR2 (0.0.2)
Maintainer: Luis Torgo <ltorgo@dcc.fc.up.pt>  
Bug reports: https://github.com/ltorgo/DMwR2/issues

0 errors | 0 warnings | 0 notes

## dplyr (0.5.0)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/dplyr/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 23.5Mb
  sub-directories of 1Mb or more:
    libs  21.4Mb
```

## eiR (1.14.1)
Maintainer: Thomas Girke <thomas.girke@ucr.edu>

1 error  | 2 warnings | 6 notes

```
checking tests ... ERROR
  Running ‘runTests.R’
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  
   
  1 Test Suite : 
  eiR RUnit Tests - 1 test function, 1 error, 0 failures
  ERROR in /tmp/RtmpXzfwIw/RLIBS_feee615d84b/eiR/unitTests/test_main.R: Error while sourcing  /tmp/RtmpXzfwIw/RLIBS_feee615d84b/eiR/unitTests/test_main.R : Error : (converted from warning) Closing open result set, pending rows
  
  Test files with failing tests
  
     test_main.R 
       /tmp/RtmpXzfwIw/RLIBS_feee615d84b/eiR/unitTests/test_main.R 
  
  
  Error in BiocGenerics:::testPackage("eiR") : 
    unit tests failed for package eiR
  Execution halted

checking for GNU extensions in Makefiles ... WARNING
Found the following file(s) containing GNU extensions:
  src/Makevars
Portable Makefiles do not use GNU extensions such as +=, :=, $(shell),
$(wildcard), ifeq ... endif. See section ‘Writing portable packages’ in
the ‘Writing R Extensions’ manual.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-33 (eiR.Rmd) 
Error: processing vignette 'eiR.Rmd' failed with diagnostics:
could not find function "pkg_ver"
Execution halted


checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’ ‘ChemmineR’ ‘RCurl’ ‘snow’
A package should be listed in only one of these fields.

checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
Non-standard file/directory found at top level:
  ‘tmp’

checking R code for possible problems ... NOTE
IddbVsIddbDist : process : <anonymous> : <anonymous>: no visible global
  function definition for ‘read.table’
checkEmbedding : embedJob: no visible global function definition for
  ‘head’
checkEmbedding: no visible global function definition for ‘head’
eiMakeDb: no visible global function definition for ‘str’
eiMakeDb: no visible global function definition for ‘read.table’
eiMakeDb: no visible global function definition for ‘cmdscale’
eiMakeDb: no visible global function definition for ‘write.table’
... 17 lines ...
toFile : writePart: no visible global function definition for
  ‘write.table’
writeIddbFile: no visible global function definition for ‘write.table’
Undefined global functions or variables:
  aggregate cmdscale head postgresqlWriteTable read.table str write.csv
  write.table
Consider adding
  importFrom("stats", "aggregate", "cmdscale")
  importFrom("utils", "head", "read.table", "str", "write.csv",
             "write.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'addTransform.Rd':
  \examples lines wider than 100 characters:
                     stop(paste("unknown type for 'input', or filename does not exist. type found:",class(input)))

Rd file 'eiCluster.Rd':
  \usage lines wider than 90 characters:
                                                              distance=getDefaultDist(descriptorType),
                                                              conn=defaultConn(dir), W = 1.39564, M=19,L=10,T=30,type="cluster",linkage="sin ... [TRUNCATED]

Rd file 'eiInit.Rd':
  \usage lines wider than 90 characters:
             conn=defaultConn(dir,create=TRUE), updateByName = FALSE, cl = NULL, connSource = NULL)

These lines will be truncated in the PDF manual.

checking compiled code ... NOTE
File ‘eiR/libs/eiR.so’:
  Found ‘_ZSt4cerr’, possibly from ‘std::cerr’ (C++)
    Objects: ‘apost.o’, ‘format.o’, ‘recordfile.o’, ‘scan.o’
  Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
    Objects: ‘apost.o’, ‘format.o’, ‘profiling.o’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor the system RNG.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## eisa (1.26.0)
Maintainer: Gabor Csardi <csardi.gabor@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘GO.db’ ‘KEGG.db’ ‘MASS’ ‘biclust’ ‘igraph’ ‘xtable’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported objects imported by ':::' calls:
  ‘Category:::getDataEnv’ ‘Category:::getKeggToProbeMap’
  ‘Category:::probeToEntrezMapHelper’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
ISA : <anonymous>: no visible global function definition for ‘IQR’
ISA: no visible binding for global variable ‘var’
ISA2heatmap: no visible global function definition for ‘heatmap’
ISACHR: no visible global function definition for ‘p.adjust’
ISAEnrichment: no visible global function definition for ‘p.adjust’
ISAGO: no visible global function definition for ‘p.adjust’
ISAHTMLModules: no visible global function definition for ‘GO_dbconn’
ISAHTMLModules : go.graph: no visible global function definition for
  ‘graph.data.frame’
... 107 lines ...
  layout.reingold.tilford lines neighborhood p.adjust par png points
  read.delim remove.vertex.attribute sd segments text title
  topological.sort unfold.tree var vcount xtable xy.coords
Consider adding
  importFrom("grDevices", "dev.cur", "dev.off", "gray", "hcl", "png",
             "xy.coords")
  importFrom("graphics", "abline", "axis", "barplot", "image", "layout",
             "lines", "par", "points", "segments", "text", "title")
  importFrom("stats", "IQR", "cor", "heatmap", "p.adjust", "sd", "var")
  importFrom("utils", "read.delim")
to your NAMESPACE file.
```

## emuR (0.2.2)
Maintainer: Raphael Winkelmann <raphael@phonetik.uni-muenchen.de>  
Bug reports: https://github.com/IPS-LMU/emuR/issues

0 errors | 0 warnings | 0 notes

## ensembldb (1.6.2)
Maintainer: Johannes Rainer <johannes.rainer@eurac.edu>  
Bug reports: https://github.com/jotsetung/ensembldb/issues

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  'GenomicFeatures:::fetchChromLengthsFromEnsembl'
  'GenomicFeatures:::fetchChromLengthsFromEnsemblPlants'
  See the note in ?`:::` about the use of this operator.

checking Rd line widths ... NOTE
Rd file 'EnsDb.Rd':
  \examples lines wider than 100 characters:
     dbcon <- dbConnect(MySQL(), user = my_user, pass = my_pass, host = my_host, dbname = "ensdb_hsapiens_v75")

These lines will be truncated in the PDF manual.
```

## etl (0.3.5)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/etl/issues

0 errors | 0 warnings | 0 notes

## ETLUtils (1.3)
Maintainer: Jan Wijffels <jwijffels@bnosac.be>

0 errors | 0 warnings | 0 notes

## ExperimentHubData (1.0.0)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘Title’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘Description’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘BiocVersion’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘Genome’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
... 19 lines ...
  variable ‘RDataDateAdded’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘RDataPath’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘DispatchClass’
makeExperimentHubMetadata : <anonymous>: no visible binding for global
  variable ‘PreparerClass’
Undefined global functions or variables:
  BiocVersion Coordinate_1_based DataProvider Description DispatchClass
  Genome Maintainer PreparerClass RDataClass RDataDateAdded RDataPath
  SourceType SourceUrl SourceVersion Species TaxonomyId Title
```

## filehashSQLite (0.2-4)
Maintainer: Roger D. Peng <rpeng@jhsph.edu>

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘filehash’ ‘DBI’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘RSQLite’ ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
createSQLite: no visible global function definition for ‘dbDriver’
createSQLite: no visible global function definition for ‘dbConnect’
createSQLite: no visible global function definition for
  ‘dbUnloadDriver’
createSQLite: no visible global function definition for ‘dbGetQuery’
initializeSQLite: no visible global function definition for ‘dbDriver’
initializeSQLite: no visible global function definition for ‘dbConnect’
initializeSQLite: no visible global function definition for ‘new’
dbDelete,filehashSQLite-character: no visible global function
... 7 lines ...
  definition for ‘dbGetQuery’
dbList,filehashSQLite: no visible global function definition for
  ‘dbGetQuery’
dbMultiFetch,filehashSQLite-character: no visible global function
  definition for ‘dbGetQuery’
Undefined global functions or variables:
  dbConnect dbDriver dbGetQuery dbUnloadDriver new
Consider adding
  importFrom("methods", "new")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## frma (1.26.0)
Maintainer: Matthew N. McCall <mccallm@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’
A package should be listed in only one of these fields.

checking R code for possible problems ... NOTE
GNUSE: no visible binding for global variable ‘median’
GNUSE: no visible binding for global variable ‘quantile’
GNUSE: no visible global function definition for ‘boxplot’
barcode: no visible global function definition for ‘pnorm’
batchFit: no visible global function definition for ‘median’
coerce<-,frmaExpressionSet-ExpressionSet: no visible global function
  definition for ‘slot<-’
coerce<-,frmaExpressionSet-ExpressionSet: no visible global function
  definition for ‘slot’
... 23 lines ...
File ‘frma/R/barcode.R’:
  data(list = gsub("frma", "barcode", pkg))
File ‘frma/R/frma.R’:
  data(list = eval(vecdataname))
File ‘frma/R/frmaBatch.R’:
  data(list = eval(vecdataname))
File ‘frma/R/frmaMedPol.R’:
  data(list = eval(vecdataname))
File ‘frma/R/frmaRobReg.R’:
  data(list = eval(vecdataname))
See section ‘Good practice’ in ‘?data’.
```

## frmaTools (1.26.0)
Maintainer: Matthew N. McCall <mccallm@gmail.com>

0 errors | 0 warnings | 4 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/VectorPkg-template/data/.dummy.txt
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘oligo’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Package in Depends field not imported from: ‘affy’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
convertPlatform: no visible global function definition for
  ‘cleancdfname’
convertPlatform: no visible global function definition for ‘cdfName’
convertPlatform: no visible global function definition for ‘probeNames’
convertPlatform: no visible global function definition for ‘pmindex’
getFidMetaProbesetCore: no visible global function definition for ‘db’
getFidMetaProbesetExtended: no visible global function definition for
  ‘db’
getFidMetaProbesetFull: no visible global function definition for ‘db’
... 13 lines ...
  ‘packageVersion’
makeVectorsFeatureSet: no visible global function definition for
  ‘read.celfiles’
makeVectorsFeatureSet: no visible global function definition for
  ‘backgroundCorrect’
Undefined global functions or variables:
  ReadAffy backgroundCorrect bg.correct.rma cdfName cleancdfname db
  packageVersion pm pmindex probeNames read.celfiles xy2indices
Consider adding
  importFrom("utils", "packageVersion")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'makeVectorPackage.Rd':
  \usage lines wider than 90 characters:
       makeVectorPackage(files, batch.id, version, maintainer, species, annotation, packageName, background="rma", normalize="quantile", nor ... [TRUNCATED]
       makeVectorsAffyBatch(files, batch.id, background="rma", normalize="quantile", normVec=NULL, cdfname=NULL, file.dir=".", verbose=TRUE)
       makeVectorsFeatureSet(files, batch.id, pkgname, background="rma", normalize="quantile", normVec=NULL, file.dir=".", verbose=TRUE)

These lines will be truncated in the PDF manual.
```

## gcbd (0.2.6)
Maintainer: Dirk Eddelbuettel <edd@debian.org>

0 errors | 0 warnings | 0 notes

## GenomicFeatures (1.26.4)
Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’
  ‘AnnotationDbi’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
  ‘rtracklayer:::resourceDescription’ ‘rtracklayer:::ucscTableOutputs’
  See the note in ?`:::` about the use of this operator.
```

## genomicper (1.6)
Maintainer: Claudia Cabrera <c.cabrera@qmul.ac.uk>

0 errors | 0 warnings | 0 notes

## Genominator (1.28.0)
Maintainer: James Bullard <bullard@stat.berkeley.edu>

0 errors | 0 warnings | 4 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘GenomeGraphs’ ‘IRanges’
  Please remove these calls from your code.
'library' or 'require' call to ‘ShortRead’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘GenomeGraphs’ ‘RSQLite’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.genominator.coverage plot.genominator.goodness.of.fit
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
addPrimingWeights: no visible global function definition for
  ‘varLabels’
addPrimingWeights: no visible global function definition for
  ‘alignData’
addPrimingWeights: no visible global function definition for ‘subseq’
addPrimingWeights: no visible global function definition for ‘sread’
addPrimingWeights: no visible global function definition for
  ‘AlignedDataFrame’
addPrimingWeights: no visible global function definition for ‘pData’
... 46 lines ...
  function definition for ‘qunif’
plot.genominator.goodness.of.fit : <anonymous>: no visible global
  function definition for ‘qqplot’
Undefined global functions or variables:
  AlignedDataFrame DisplayPars alignData chromosome gdPlot
  geneRegionBiomart makeAnnotationTrack makeBaseTrack makeGenericArray
  makeGenomeAxis mkAllStrings pData position ppoints qchisq qqplot
  qunif readAligned sread subseq tables varLabels varMetadata
Consider adding
  importFrom("stats", "ppoints", "qchisq", "qqplot", "qunif")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'makeGeneRepresentation.Rd':
  \usage lines wider than 90 characters:
     "background"), gene.id = "ensembl_gene_id", transcript.id = "ensembl_transcript_id", bind.columns, ignoreStrand = TRUE, verbose = getOp ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## GenVisR (1.4.1)
Maintainer: Zachary Skidmore <zlskidmore@gmail.com>  
Bug reports: https://github.com/griffithlab/GenVisR/issues

0 errors | 0 warnings | 0 notes

## gmDatabase (0.5.0)
Maintainer: K. Gerald van den Boogaart <support@boogaart.de>

0 errors | 0 warnings | 0 notes

## GOFunction (1.22.0)
Maintainer: Jing Wang <jingwang.uestc@gmail.com>

0 errors | 1 warning  | 5 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘GOFunction.pdf’ from 679Kb to 143Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘graph’ ‘Rgraphviz’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’ ‘Biobase’ ‘graph’ ‘Rgraphviz’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘GO.db’ ‘Rgraphviz’ ‘SparseM’ ‘graph’ ‘methods’
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘GO.db’ ‘Rgraphviz’ ‘graph’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
File ‘GOFunction/R/zzz.R’:
  .onLoad calls:
    require(methods)

Package startup functions should not change the search path.
See section ‘Good practice’ in '?.onAttach'.

GOFunction: no visible global function definition for ‘nodes’
GOFunction: no visible global function definition for ‘write.csv’
... 15 lines ...
showSigNodes: no visible global function definition for ‘plot’
showSigNodes: no visible global function definition for ‘dev.off’
Undefined global functions or variables:
  bmp dev.off edgeWeights getDefaultAttrs graph2SparseM heat.colors
  nodes p.adjust phyper plot sparseM2Graph write.csv
Consider adding
  importFrom("grDevices", "bmp", "dev.off", "heat.colors")
  importFrom("graphics", "plot")
  importFrom("stats", "p.adjust", "phyper")
  importFrom("utils", "write.csv")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'GOFunction.Rd':
  \usage lines wider than 90 characters:
     GOFunction(interestGenes, refGenes, organism = "org.Hs.eg.db", ontology = "BP", fdrmethod = "BY", fdrth = 0.05, ppth =   0.05, pcth = 0 ... [TRUNCATED]
  \examples lines wider than 100 characters:
            sigTerm <- GOFunction(interestGenes, refGenes, organism = "org.Hs.eg.db", ontology= "BP", fdrmethod = "BY", 
            fdrth = 0.05, ppth = 0.05, pcth = 0.05, poth = 0.05, peth = 0.05, bmpSize = 2000, filename="sigTerm")

These lines will be truncated in the PDF manual.
```

## GWASTools (1.20.0)
Maintainer: Stephanie M. Gogarten <sdmorris@u.washington.edu>, Adrienne Stilp <amstilp@u.washington.edu>

0 errors | 0 warnings | 1 note 

```
checking Rd line widths ... NOTE
Rd file 'assocRegression.Rd':
  \examples lines wider than 100 characters:
     scanAnnot$blood.pressure[scanAnnot$case.cntl.status==1] <- rnorm(sum(scanAnnot$case.cntl.status==1), mean=100, sd=10)
     scanAnnot$blood.pressure[scanAnnot$case.cntl.status==0] <- rnorm(sum(scanAnnot$case.cntl.status==0), mean=90, sd=5)

Rd file 'createDataFile.Rd':
  \usage lines wider than 90 characters:
                    precision="single", compress="ZIP_RA:8M", compress.geno="ZIP_RA", compress.annot="ZIP_RA",

These lines will be truncated in the PDF manual.
```

## implyr (0.1.0)
Maintainer: Ian Cook <ian@cloudera.com>  
Bug reports: https://github.com/ianmcook/implyr/issues

0 errors | 0 warnings | 0 notes

## imputeMulti (0.6.4)
Maintainer: Alex Whitworth <whitworth.alex@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking dependencies in R code ... NOTE
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘count_compare’
```

## isobar (1.20.0)
Maintainer: Florian P Breitwieser <florian.bw@gmail.com>  
Bug reports: https://github.com/fbreitwieser/isobar/issues

0 errors | 0 warnings | 5 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘BinaryFiles’

checking dependencies in R code ... NOTE
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘.as.matrix’ ‘.as.vect’ ‘.convertPeptideModif’
  ‘.proteinGroupAsConciseDataFrame’ ‘.read.idfile’ ‘.sum.bool’

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  as.data.frame.IBSpectra as.data.frame.ProteinGroup plot.NoiseModel
  summary.ProteinGroup
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
.calcProbXGreaterThanY.orig: no visible global function definition for
  ‘d’
.read.peaklist: no visible binding for global variable ‘type’
.round.distr: no visible global function definition for ‘param’
.write.summarized.table: no visible binding for global variable ‘name’
distrprint: no visible global function definition for ‘param’
distrprint : <anonymous>: no visible global function definition for
  ‘param’
shared.ratios.sign: no visible binding for global variable ‘ratio’
... 21 lines ...
  for global variable ‘noise.model.col’
plotRatio,IBSpectra-character-character-character: no visible binding
  for global variable ‘pch.p’
scale,Tlsd: no visible global function definition for ‘param’
Undefined global functions or variables:
  center.var d g i mz name noise.model.col o param pch pch.p peptide
  qual ratio type

Found the following assignments to the global environment:
File ‘isobar/R/report-utils.R’:
  assign("properties.env", properties.env, envir = .GlobalEnv)

checking Rd line widths ... NOTE
Rd file 'calculate.emPAI.Rd':
  \usage lines wider than 90 characters:
     calculate.emPAI(protein.group, protein.g = reporterProteins(protein.group), normalize = FALSE,
                     observed.pep = c("pep", "mod.charge.pep"), use.mw = FALSE, combine.f = mean, 
                                       mass = c(164.554862, 278.61037, 213.12392, 150.953636)), ...)

Rd file 'distr-methods.Rd':
  \usage lines wider than 90 characters:
       calcProbXDiffNormals(X, mu_Y, sd_Y, ..., alternative = c("greater", "less", "two-sided"), progress = FALSE)
... 47 lines ...
     proteinRatios(ibspiked_set1,noise.model=noise.model.hcd,proteins=ceru.proteins,cl=c("T","T","C","C"),combn.method="interclass",summariz ... [TRUNCATED]

Rd file 'shared.ratios.Rd':
  \usage lines wider than 90 characters:
     shared.ratios(ibspectra, noise.model, channel1 , channel2 , protein = reporterProteins(proteinGroup(ibspectra)), ...)

Rd file 'spectra.count2.Rd':
  \usage lines wider than 90 characters:
                    type = "protein.g", specificity = c("reporter-specific", "group-specific", "unspecific"), 

These lines will be truncated in the PDF manual.
```

## ITALICS (2.34.0)
Maintainer: Guillem Rigaill <italics@curie.fr>

0 errors | 0 warnings | 5 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘affxparser’ ‘GLAD’ ‘oligo’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘ITALICSData’ ‘pd.mapping50k.xba240’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
trainITALICS: warning in list.files(dir, full.name = TRUE): partial
  argument match of 'full.name' to 'full.names'

checking Rd files ... NOTE
prepare_Rd: GLAD.GLAD.Rd:33: Dropping empty section \keyword
prepare_Rd: ITALICSNew.Rd:82: Dropping empty section \keyword
prepare_Rd: ITALICSTrain.Rd:43: Dropping empty section \keyword
prepare_Rd: Model.GetConfDat.Rd:23: Dropping empty section \keyword
prepare_Rd: Model.GetCorrection.Rd:20: Dropping empty section \keyword
prepare_Rd: Model.GetEffet.Rd:20: Dropping empty section \keyword
prepare_Rd: Model.GetModel.Rd:21: Dropping empty section \keyword
prepare_Rd: Model.GetResidu.Rd:18: Dropping empty section \keyword
prepare_Rd: New.AddInfo.Rd:24: Dropping empty section \keyword
prepare_Rd: New.fromQuartetToSnp.Rd:31: Dropping empty section \keyword
prepare_Rd: New.fromQuartetToSnp.Rd:25: Dropping empty section \examples
prepare_Rd: New.fromSnpToQuartet.Rd:25: Dropping empty section \keyword
prepare_Rd: New.getQuartet.Rd:27: Dropping empty section \keyword
prepare_Rd: New.getSnpInfo.Rd:25: Dropping empty section \keyword
prepare_Rd: New.readQuartetCopyNb.Rd:23: Dropping empty section \keyword

checking Rd line widths ... NOTE
Rd file 'ITALICSNew.Rd':
  \usage lines wider than 90 characters:
         amplicon=2.1, deletion=-3.5, deltaN=0.15, forceGL=c(-0.2,0.2), param=c(d=2), nbsigma=1, ... )
  \examples lines wider than 100 characters:
     ## load annotation using the pd.mapping50k.xba24 or pd.mapping50k.hind240 or  pd.mapping250k.sty or pd.mapping250k.nsp package
     snpInfo <- fromQuartetToSnp(cIntensity="quartetLogRatio", quartetInfo=quartet$quartetInfo, snpInfo=snpInfo)

Rd file 'ITALICSTrain.Rd':
  \usage lines wider than 90 characters:
     trainITALICS (dir,  amplicon=2.1, deletion=-3.5, deltaN=0.15, forceGL=c(-0.2,0.2), param=c(d=2), nbsigma=1, ...)

These lines will be truncated in the PDF manual.
```

## keggorthology (2.26.0)
Maintainer: VJ Carey <stvjc@channing.harvard.edu>

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘graph’ ‘stats’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘hgu95av2.db’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
getKOprobes: no visible binding for global variable ‘KOgraph’
getKOtags: no visible binding for global variable ‘KOgraph’
Undefined global functions or variables:
  KOgraph

Found the following calls to data() loading into the global environment:
File ‘keggorthology/R/getKOtags.R’:
  data(KOgraph)
  data(KOgraph)
See section ‘Good practice’ in ‘?data’.
```

## knitr (1.15.1)
Maintainer: Yihui Xie <xie@yihui.name>  
Bug reports: https://github.com/yihui/knitr/issues

0 errors | 0 warnings | 0 notes

## liteq (1.0.0)
Maintainer: Gábor Csárdi <csardi.gabor@gmail.com>  
Bug reports: https://github.com/gaborcsardi/liteq/issues

0 errors | 0 warnings | 0 notes

## lumi (2.26.4)
Maintainer: Pan Du <dupan.mail@gmail.com>, Lei Huang <lhuang@bsd.uchicago.edu>, Gang Feng <g-feng@northwestern.edu>

0 errors | 1 warning  | 3 notes

```
checking dependencies in R code ... WARNING
'::' or ':::' imports not declared from:
  ‘IRanges’ ‘bigmemoryExtras’

checking installed package size ... NOTE
  installed size is  6.8Mb
  sub-directories of 1Mb or more:
    R      2.8Mb
    data   3.6Mb

checking Rd line widths ... NOTE
Rd file 'IlluminaID2nuID.Rd':
  \usage lines wider than 90 characters:
     IlluminaID2nuID(IlluminaID, lib.mapping=NULL, species = c("Human", "Mouse", "Rat", "Unknown"), chipVersion = NULL, ...)

Rd file 'addAnnotationInfo.Rd':
  \usage lines wider than 90 characters:
     addAnnotationInfo(methyLumiM, lib = 'FDb.InfiniumMethylation.hg19', annotationColumn=c('COLOR_CHANNEL', 'CHROMOSOME', 'POSITION'))

Rd file 'addNuID2lumi.Rd':
... 177 lines ...
     smoothQuantileNormalization(dataMatrix, ref = NULL, adjData=NULL, logMode = TRUE, bandwidth = NULL, degree = 1, verbose = FALSE, ...)

Rd file 'ssn.Rd':
  \usage lines wider than 90 characters:
     ssn(x.lumi, targetArray = NULL, scaling = TRUE, bgMethod=c('density', 'mean', 'median', 'none'), fgMethod=c('mean', 'density', 'median' ... [TRUNCATED]

Rd file 'vst.Rd':
  \usage lines wider than 90 characters:
     vst(u, std, nSupport = min(length(u), 500), backgroundStd=NULL, fitMethod = c('linear', 'quadratic'), lowCutoff = 1/3, ifPlot = FALSE)

These lines will be truncated in the PDF manual.

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘bigmemoryExtras’
```

## macleish (0.3.0)
Maintainer: Ben Baumer <ben.baumer@gmail.com>

0 errors | 0 warnings | 0 notes

## marmap (0.9.6)
Maintainer: Eric Pante <pante.eric@gmail.com>  
Bug reports: https://github.com/ericpante/marmap/issues

0 errors | 0 warnings | 0 notes

## mdgsa (1.6.0)
Maintainer: David Montaner <dmontaner@cipf.es>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking R code for possible problems ... NOTE
getGOnames: no visible global function definition for
  ‘packageDescription’
getKEGGnames: no visible global function definition for
  ‘packageDescription’
getOntology: no visible global function definition for
  ‘packageDescription’
goLeaves: no visible global function definition for
  ‘packageDescription’
index.normalize: no visible global function definition for ‘qqnorm’
... 19 lines ...
uvGsa: no visible global function definition for ‘summary.glm’
uvGsa: no visible global function definition for ‘p.adjust’
Undefined global functions or variables:
  abline cov glm.fit lines p.adjust packageDescription plot points
  qchisq qqnorm quasibinomial read.table sd summary.glm
Consider adding
  importFrom("graphics", "abline", "lines", "plot", "points")
  importFrom("stats", "cov", "glm.fit", "p.adjust", "qchisq", "qqnorm",
             "quasibinomial", "sd", "summary.glm")
  importFrom("utils", "packageDescription", "read.table")
to your NAMESPACE file.
```

## mdsr (0.1.3)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/mdsr/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.4Mb
  sub-directories of 1Mb or more:
    data   5.3Mb
```

## MetaIntegrator (1.0.3)
Maintainer: Winston A. Haynes <hayneswa@stanford.edu>

0 errors | 0 warnings | 0 notes

## mgsa (1.22.0)
Maintainer: Sebastian Bauer <mail@sebastianbauer.info>

0 errors | 1 warning  | 4 notes

```
checking for GNU extensions in Makefiles ... WARNING
Found the following file(s) containing GNU extensions:
  src/Makevars
  src/Makevars.in
Portable Makefiles do not use GNU extensions such as +=, :=, $(shell),
$(wildcard), ifeq ... endif. See section ‘Writing portable packages’ in
the ‘Writing R Extensions’ manual.

checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘acinclude.m4’ ‘aclocal.m4’ ‘script’

checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘mgsa’ for: ‘show’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘gplots’ which was already attached by Depends.
  Please remove these calls from your code.
'library' or 'require' calls in package code:
  ‘DBI’ ‘GO.db’ ‘RSQLite’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘gplots’ ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
createMgsaGoSets: no visible global function definition for ‘new’
mcmcSummary: no visible binding for global variable ‘sd’
mgsa.wrapper: no visible global function definition for ‘str’
mgsa.wrapper: no visible global function definition for ‘new’
readGAF: no visible global function definition for ‘read.delim’
readGAF: no visible global function definition for ‘na.omit’
readGAF: no visible global function definition for ‘new’
initialize,MgsaSets: no visible global function definition for
  ‘callNextMethod’
... 10 lines ...
  ‘close.screen’
Undefined global functions or variables:
  barplot2 callNextMethod close.screen na.omit new par read.delim
  relist screen sd split.screen str
Consider adding
  importFrom("graphics", "close.screen", "par", "screen", "split.screen")
  importFrom("methods", "callNextMethod", "new")
  importFrom("stats", "na.omit", "sd")
  importFrom("utils", "read.delim", "relist", "str")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## miRNAmeConverter (1.2.0)
Maintainer: Stefan J. Haunsberger <stefan.haunsberger@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
MiRNANameConverter,ANY: no visible global function definition for ‘new’
Undefined global functions or variables:
  new
Consider adding
  importFrom("methods", "new")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## miRNAtap (1.8.0)
Maintainer: Maciej Pajak <m.pajak@sms.ed.ac.uk>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘miRNAtap.db’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
getTargetsFromSource: no visible binding for global variable
  ‘miRNAtap.db’
getTargetsFromSource : <anonymous>: no visible binding for global
  variable ‘miRNAtap.db’
translate: no visible binding for global variable ‘miRNAtap.db’
Undefined global functions or variables:
  miRNAtap.db
```

## mitools (2.3)
Maintainer: Thomas Lumley <t.lumley@auckland.ac.nz>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
MIcombine.default: no visible binding for global variable ‘vcov’
MIcombine.default: no visible binding for global variable ‘coef’
MIcombine.default: no visible global function definition for ‘var’
MIcombine.imputationResultList: no visible binding for global variable
  ‘vcov’
MIcombine.imputationResultList: no visible binding for global variable
  ‘coef’
close.DBimputationList: no visible global function definition for ‘is’
getvars: no visible global function definition for ‘is’
print.MIresult: no visible global function definition for ‘coef’
print.MIresult: no visible global function definition for ‘vcov’
summary.MIresult: no visible global function definition for ‘coef’
summary.MIresult: no visible global function definition for ‘vcov’
summary.MIresult: no visible global function definition for ‘qt’
Undefined global functions or variables:
  coef is qt var vcov
Consider adding
  importFrom("methods", "is")
  importFrom("stats", "coef", "qt", "var", "vcov")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## MmPalateMiRNA (1.24.0)
Maintainer: Guy Brock <guy.brock@louisville.edu>

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘xtable’ ‘limma’ ‘statmod’ ‘lattice’ ‘vsn’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘limma’ ‘lattice’ ‘Biobase’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘methods’ ‘statmod’ ‘vsn’ ‘xtable’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
clustPlot: no visible global function definition for ‘par’
clustPlot: no visible global function definition for ‘plot’
clustPlot: no visible global function definition for ‘axis’
clustPlot: no visible global function definition for ‘lines’
clustPlot: no visible global function definition for ‘legend’
fixOutliers : <anonymous>: no visible global function definition for
  ‘sd’
imputeKNN: no visible global function definition for ‘sd’
imputeKNN: no visible global function definition for ‘cor’
... 36 lines ...
  definition for ‘assayData’
levelplot,list-missing: no visible global function definition for
  ‘median’
Undefined global functions or variables:
  RG.MA as.formula assayData axis cor featureData legend lines mad
  median pData par plot sd stack
Consider adding
  importFrom("graphics", "axis", "legend", "lines", "par", "plot")
  importFrom("stats", "as.formula", "cor", "mad", "median", "sd")
  importFrom("utils", "stack")
to your NAMESPACE file.
```

## MonetDBLite (0.3.1)
Maintainer: Hannes Muehleisen <hannes@cwi.nl>  
Bug reports: https://github.com/hannesmuehleisen/MonetDBLite/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  7.0Mb
  sub-directories of 1Mb or more:
    libs   6.7Mb
```

## MonetDB.R (1.0.1)
Maintainer: Hannes Muehleisen <hannes@cwi.nl>

0 errors | 0 warnings | 0 notes

## oai (0.2.2)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/oai/issues

0 errors | 0 warnings | 0 notes

## oce (0.9-21)
Maintainer: Dan Kelley <Dan.Kelley@Dal.Ca>  
Bug reports: https://github.com/dankelley/oce/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘oce-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: read.oce
> ### Title: Read an Oceanographic Data File
> ### Aliases: read.oce
> 
> ### ** Examples
> 
> 
> library(oce)
> x <- read.oce(system.file("extdata", "ctd.cnv", package="oce"))
> plot(x) # summary with TS and profiles
Error in if (!is.null(x@metadata$startTime) && 4 < nchar(x@metadata$startTime,  : 
  missing value where TRUE/FALSE needed
Calls: plot -> plot -> .local
Execution halted

checking installed package size ... NOTE
  installed size is  5.4Mb
  sub-directories of 1Mb or more:
    help   2.1Mb
```

## odbc (1.0.1)
Maintainer: Jim Hester <james.hester@rstudio.com>  
Bug reports: https://github.com/rstats-db/odbc/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘odbc’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/DBI/revdep/checks/odbc.Rcheck/00install.out’ for details.
```

## ODB (1.1.1)
Maintainer: Sylvain Mareschal <maressyl@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘DBI’ ‘RJDBC’ ‘methods’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
isClosed: no visible global function definition for ‘dbSendUpdate’
odb.close: no visible global function definition for ‘is’
odb.close: no visible global function definition for ‘validObject’
odb.close: no visible global function definition for ‘dbSendUpdate’
odb.close: no visible global function definition for ‘dbDisconnect’
odb.comments: no visible global function definition for ‘is’
odb.comments: no visible global function definition for ‘validObject’
odb.comments<-: no visible global function definition for ‘is’
odb.comments<-: no visible global function definition for ‘validObject’
... 31 lines ...
  ‘flush.console’
set,progress.file: no visible global function definition for
  ‘flush.console’
Undefined global functions or variables:
  JDBC dbColumnInfo dbConnect dbDisconnect dbListTables dbSendQuery
  dbSendUpdate fetch flush.console is new tail unzip validObject
Consider adding
  importFrom("methods", "is", "new", "validObject")
  importFrom("utils", "flush.console", "tail", "unzip")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## oligo (1.38.0)
Maintainer: Benilton Carvalho <benilton@unicamp.br>

1 error  | 0 warnings | 8 notes

```
checking examples ... ERROR
Running examples in ‘oligo-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: MAplot
> ### Title: MA plots
> ### Aliases: MAplot MAplot-methods MAplot,FeatureSet-method
> ###   MAplot,TilingFeatureSet-method MAplot,PLMset-method
> ###   MAplot,ExpressionSet-method MAplot,matrix-method
... 8 lines ...
+   groups <- factor(rep(c('brain', 'UnivRef'), each=3))
+   data.frame(sampleNames(nimbleExpressionFS), groups)
+   MAplot(nimbleExpressionFS, pairs=TRUE, ylim=c(-.5, .5), groups=groups)
+ }
Loading required package: oligoData
Loading required package: pd.hg18.60mer.expr
Loading required package: RSQLite
Loading required package: DBI
Error in loadNamespace(name) : there is no package called ‘KernSmooth’
Calls: MAplot ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted

checking installed package size ... NOTE
  installed size is 30.4Mb
  sub-directories of 1Mb or more:
    R         1.2Mb
    doc      12.9Mb
    scripts  15.7Mb

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘biomaRt’ ‘AnnotationDbi’ ‘GenomeGraphs’ ‘RCurl’ ‘ff’
A package should be listed in only one of these fields.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘TODO.org’

checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘oligo’ for: ‘show’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘Biobase:::annotatedDataFrameFromMatrix’
  See the note in ?`:::` about the use of this operator.

checking foreign function calls ... NOTE
Foreign function calls to a different package:
  .Call("ReadHeader", ..., PACKAGE = "affyio")
  .Call("read_abatch", ..., PACKAGE = "affyio")
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.

checking R code for possible problems ... NOTE
image,FeatureSet: warning in matrix(NA, nr = geom[1], nc = geom[2]):
  partial argument match of 'nr' to 'nrow'
image,FeatureSet: warning in matrix(NA, nr = geom[1], nc = geom[2]):
  partial argument match of 'nc' to 'ncol'
NUSE: no visible global function definition for ‘abline’
RLE: no visible global function definition for ‘abline’
basicMvApairsPlot: no visible binding for global variable
  ‘smoothScatter’
basicMvApairsPlot: no visible global function definition for ‘frame’
... 36 lines ...
Undefined global functions or variables:
  IQR abline aggregate approx complete.cases data frame intensities
  loess man_fsetid mtext predict rnorm smooth.spline smoothScatter
  splinefun text
Consider adding
  importFrom("graphics", "abline", "frame", "mtext", "smoothScatter",
             "text")
  importFrom("stats", "IQR", "aggregate", "approx", "complete.cases",
             "loess", "predict", "rnorm", "smooth.spline", "splinefun")
  importFrom("utils", "data")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'basicRMA.Rd':
  \usage lines wider than 90 characters:
     basicRMA(pmMat, pnVec, normalize = TRUE, background = TRUE, bgversion = 2, destructive = FALSE, verbose = TRUE, ...)

Rd file 'fitProbeLevelModel.Rd':
  \usage lines wider than 90 characters:
     fitProbeLevelModel(object, background=TRUE, normalize=TRUE, target="core", method="plm", verbose=TRUE, S4=TRUE, ...)

Rd file 'getProbeInfo.Rd':
  \usage lines wider than 90 characters:
     getProbeInfo(object, field, probeType = "pm", target = "core", sortBy = c("fid", "man_fsetid", "none"), ...)
  \examples lines wider than 100 characters:
        agenGene <- getProbeInfo(affyGeneFS, field=c('fid', 'fsetid', 'type'), target='probeset', subset= type == 'control->bgp->antigenomic ... [TRUNCATED]

Rd file 'preprocessTools.Rd':
  \usage lines wider than 90 characters:
     backgroundCorrect(object, method=backgroundCorrectionMethods(), copy=TRUE, extra, subset=NULL, target='core', verbose=TRUE)
     normalize(object, method=normalizationMethods(), copy=TRUE, subset=NULL,target='core', verbose=TRUE, ...)

These lines will be truncated in the PDF manual.
```

## ora (2.0-1)
Maintainer: Arni Magnusson <arnima@hafro.is>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘ROracle’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## PAnnBuilder (1.38.0)
Maintainer: Li Hong <sysptm@gmail.com>

0 errors | 3 warnings | 1 note 

```
checking dependencies in R code ... WARNING
'library' or 'require' call to ‘Biobase’ which was already attached by Depends.
  Please remove these calls from your code.
':::' calls which should be '::':
  ‘AnnotationDbi:::as.list’ ‘base:::get’ ‘tools:::list_files_with_type’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by ':::' calls:
  ‘AnnotationDbi:::createAnnDbBimaps’
  ‘AnnotationDbi:::prefixAnnObjNames’ ‘tools:::makeLazyLoadDB’
  See the note in ?`:::` about the use of this operator.
  Including base/recommended package(s):
  ‘tools’
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘getShortSciName’ ‘twoStepSplit’

checking R code for possible problems ... WARNING
Found an obsolete/platform-specific call in the following function:
  ‘makeLLDB’
Found the defunct/removed function:
  ‘.saveRDS’

In addition to the above warning(s), found the following notes:

File ‘PAnnBuilder/R/zzz.R’:
  .onLoad calls:
    require(Biobase)

Package startup functions should not change the search path.
See section ‘Good practice’ in '?.onAttach'.

Found the following calls to data() loading into the global environment:
File ‘PAnnBuilder/R/writeManPage.R’:
  data("descriptionInfo")
  data("descriptionInfo")
See section ‘Good practice’ in ‘?data’.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
             }
     

trying URL 'http://gpcr2.biocomp.unibo.it/bacello/dataset.htm'
Content type 'text/html; charset=iso-8859-1' length 5062 bytes
==================================================
downloaded 5062 bytes
... 8 lines ...
Warning in rsqlite_disconnect(conn@ptr) :
  There are 1 result in use. The connection will be released when they are closed
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'PAnnBuilder.tex' failed.
LaTeX errors:
! Package auto-pst-pdf Error: 
    "shell escape" (or "write18") is not enabled:
    auto-pst-pdf will not work!
.
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘methods’ ‘utils’ ‘Biobase’ ‘RSQLite’ ‘AnnotationDbi’
A package should be listed in only one of these fields.
```

## pdInfoBuilder (1.38.0)
Maintainer: Benilton Carvalho <beniltoncarvalho@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
cdf2table: no visible global function definition for ‘getDoParWorkers’
cdf2table: no visible global function definition for ‘%dopar%’
cdf2table: no visible global function definition for ‘foreach’
cdf2table: no visible binding for global variable ‘unitLst’
cdfUnits2table: no visible global function definition for ‘%do%’
cdfUnits2table: no visible global function definition for ‘foreach’
cdfUnits2table: no visible binding for global variable ‘i’
createChrDict: no visible global function definition for ‘na.omit’
getAllFSetMpsTables: no visible global function definition for
  ‘%dopar%’
getAllFSetMpsTables: no visible global function definition for
  ‘foreach’
getAllFSetMpsTables: no visible binding for global variable ‘i’
parseBpmapCel: no visible global function definition for ‘aggregate’
parseCdfSeqAnnotSnp: no visible global function definition for
  ‘aggregate’
parseNgsTrio: no visible global function definition for ‘aggregate’
Undefined global functions or variables:
  %do% %dopar% aggregate foreach getDoParWorkers i na.omit unitLst
Consider adding
  importFrom("stats", "aggregate", "na.omit")
to your NAMESPACE file.
```

## pitchRx (1.8.2)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: http://github.com/cpsievert/pitchRx/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘ggsubplot’
```

## PivotalR (0.1.18.3)
Maintainer: Rahul Iyer <riyer@pivotal.io>

0 errors | 0 warnings | 0 notes

## plethy (1.12.1)
Maintainer: Daniel Bottomly <bottomly@ohsu.edu>

2 errors | 0 warnings | 3 notes

```
checking examples ... ERROR
Running examples in ‘plethy-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: Utility functions
> ### Title: Utility functions to assist with QA/QC and analysis of
> ###   plethysmography data
> ### Aliases: add.labels.by.sample get.err.breaks adjust.labels proc.sanity
> ### Keywords: Utilities
... 21 lines ...
> temp.db.file <- tempfile()
> write(sim.bux.lines, file=temp.file)
> test.bux.db <- parse.buxco(file.name=temp.file, db.name=temp.db.file, chunk.size=10000)
Processing /tmp/RtmpcgUFtA/fileb5115349d16c in chunks of 10000
Starting chunk 1
Reached breakpoint change
Processing breakpoint 1
Starting sample sample_1
Error in if (sum(which.gt) > 0) { : missing value where TRUE/FALSE needed
Calls: parse.buxco ... write.sample.breaks -> write.sample.db -> sanity.check.time
Execution halted

checking tests ... ERROR
  Running ‘runTests.R’ [23s/23s]
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  
  Test files with failing tests
  
     test_check_helpers.R 
       test.add.labels.by.sample 
       test.dbImport 
       test.get.err.breaks 
       test.summaryMeasures 
  
  
  Error in BiocGenerics:::testPackage("plethy") : 
    unit tests failed for package plethy
  In addition: Warning message:
  closing unused connection 3 (/tmp/RtmpxILgv7/fileb6eaa4b1163) 
  Execution halted

checking dependencies in R code ... NOTE
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘csv.to.table’ ‘find.break.ranges.integer’ ‘fix.time’ ‘multi.grep’

checking R code for possible problems ... NOTE
generate.sample.buxco : <anonymous> : <anonymous> : <anonymous> :
  <anonymous>: no visible global function definition for ‘rnorm’
make.db.package: no visible global function definition for
  ‘packageDescription’
mvtsplot.data.frame: no visible global function definition for ‘colors’
mvtsplot.data.frame: no visible global function definition for ‘par’
mvtsplot.data.frame: no visible global function definition for ‘layout’
mvtsplot.data.frame: no visible global function definition for
  ‘strwidth’
... 14 lines ...
tsplot,BuxcoDB: no visible binding for global variable ‘Sample_Name’
Undefined global functions or variables:
  Axis Days Sample_Name Value abline bxp colors layout legend lines
  median mtext packageDescription par plot rnorm strwidth terms
Consider adding
  importFrom("grDevices", "colors")
  importFrom("graphics", "Axis", "abline", "bxp", "layout", "legend",
             "lines", "mtext", "par", "plot", "strwidth")
  importFrom("stats", "median", "rnorm", "terms")
  importFrom("utils", "packageDescription")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'parsing.Rd':
  \usage lines wider than 90 characters:
     parse.buxco(file.name = NULL, table.delim = "Table", burn.in.lines = c("Measurement", "Create measurement", "Waiting for", "Site Acknow ... [TRUNCATED]
       chunk.size = 500, db.name = "bux_test.db", max.run.time.minutes = 60, overwrite = TRUE, verbose=TRUE, make.package = F, author = NULL ... [TRUNCATED]
     parse.buxco.basic(file.name=NULL, table.delim="Table", burn.in.lines=c("Measurement", "Create measurement", "Waiting for", "Site Acknow ... [TRUNCATED]

Rd file 'utilities.Rd':
  \usage lines wider than 90 characters:
     get.err.breaks(bux.db, max.exp.count=150, max.acc.count=900, vary.perc=.1, label.val="ERR")
     proc.sanity(bux.db, max.exp.time=300, max.acc.time=1800, max.exp.count=150, max.acc.count=900)
  \examples lines wider than 100 characters:
     err.dta <- data.frame(samples=samples, count=count, measure_break=measure_break, table_break=table_break, phase=phase, stringsAsFactors ... [TRUNCATED]
     sample.labels <- data.frame(samples=c("sample_1","sample_3"), response_type=c("high", "low"),stringsAsFactors=FALSE)

These lines will be truncated in the PDF manual.
```

## poplite (0.99.17.3)
Maintainer: Daniel Bottomly <bottomly@ohsu.edu>

0 errors | 0 warnings | 0 notes

## postGIStools (0.2.1)
Maintainer: Philippe Marchand <pmarchand@sesync.org>  
Bug reports: https://github.com/SESYNC-ci/postGIStools/issues

0 errors | 0 warnings | 0 notes

## ProjectTemplate (0.7)
Maintainer: Kenton White <jkentonwhite@gmail.com>  
Bug reports: https://github.com/johnmyleswhite/ProjectTemplate/issues

0 errors | 0 warnings | 0 notes

## quantmod (0.4-7)
Maintainer: Joshua M. Ulrich <josh.m.ulrich@gmail.com>  
Bug reports: https://github.com/joshuaulrich/quantmod/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Found the following calls to attach():
File ‘quantmod/R/attachSymbols.R’:
  attach(NULL, pos = pos, name = DB$name)
  attach(NULL, pos = pos, name = DB$name)
See section ‘Good practice’ in ‘?attach’.
```

## RecordLinkage (0.4-10)
Maintainer: Andreas Borg <borga@uni-mainz.de>

0 errors | 0 warnings | 0 notes

## redcapAPI (1.3)
Maintainer: Stephen Lane <stephen.lane@barwonhealth.org.au>  
Bug reports: https://github.com/nutterb/redcapAPI/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
allocationTable_offline: no visible global function definition for
  ‘read.csv’
exportArms.redcapApiConnection: no visible global function definition
  for ‘read.csv’
exportEvents.redcapApiConnection: no visible global function definition
  for ‘read.csv’
exportInstruments.redcapApiConnection: no visible global function
  definition for ‘read.csv’
exportMappings.redcapApiConnection: no visible global function
... 21 lines ...
  definition for ‘write.table’
validateImport : printLog: no visible global function definition for
  ‘write.table’
validateImport: no visible binding for global variable ‘tail’
validateImport: no visible binding for global variable ‘head’
Undefined global functions or variables:
  capture.output head read.csv tail write.table
Consider adding
  importFrom("utils", "capture.output", "head", "read.csv", "tail",
             "write.table")
to your NAMESPACE file.
```

## refGenome (1.7.3)
Maintainer: Wolfgang Kaisers <kaisers@med.uni-duesseldorf.de>

0 errors | 0 warnings | 0 notes

## RImmPort (1.2.0)
Maintainer: Ravi Shankar <rshankar@stanford.edu>

0 errors | 1 warning  | 1 note 

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘RImmPort_Article.pdf’ from 731Kb to 336Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking R code for possible problems ... NOTE
getCellularQuantification: no visible binding for global variable
  ‘experiment_sample_accession’
getCellularQuantification: no visible binding for global variable
  ‘control_files_names’
getCellularQuantification: no visible binding for global variable
  ‘ZBREFIDP’
getGeneticsFindings: no visible binding for global variable
  ‘experiment_sample_accession’
getNucleicAcidQuantification: no visible binding for global variable
  ‘experiment_sample_accession’
getProteinQuantification: no visible binding for global variable
  ‘experiment_sample_accession’
getTiterAssayResults: no visible binding for global variable
  ‘experiment_sample_accession’
Undefined global functions or variables:
  ZBREFIDP control_files_names experiment_sample_accession
```

## RJDBC (0.2-5)
Maintainer: Simon Urbanek <Simon.Urbanek@r-project.org>

0 errors | 1 warning  | 1 note 

```
checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic 'dbReadTable' and siglist 'JDBCConnection,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘java-src’
```

## RmiR (1.30.0)
Maintainer: Francesco Favero <favero.francesco@gmail.com>

0 errors | 0 warnings | 5 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘RSVGTipsDevice’ which was already attached by Depends.
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘RSVGTipsDevice’ ‘RmiR.Hs.miRNA’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
plotRmiRtc: no visible global function definition for ‘devSVGTips’
plotRmiRtc: no visible global function definition for ‘plot’
plotRmiRtc: no visible global function definition for
  ‘setSVGShapeToolTip’
plotRmiRtc: no visible global function definition for ‘setSVGShapeURL’
plotRmiRtc: no visible global function definition for ‘points’
plotRmiRtc: no visible global function definition for ‘lines’
plotRmiRtc: no visible global function definition for ‘legend’
plotRmiRtc: no visible global function definition for ‘dev.off’
readRmiRtc: no visible global function definition for ‘write.table’
Undefined global functions or variables:
  dev.off devSVGTips legend lines plot points setSVGShapeToolTip
  setSVGShapeURL write.table
Consider adding
  importFrom("grDevices", "dev.off")
  importFrom("graphics", "legend", "lines", "plot", "points")
  importFrom("utils", "write.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'RmiR.Rd':
  \usage lines wider than 90 characters:
       RmiR(mirna=NULL,genes=NULL,annotation=NULL,dbname="targetscan",org="Hs",id="probes",id.out="symbol",verbose=FALSE)
     read.mir(mirna=NULL,genes=NULL,annotation=NULL,id="probes",dbname=c("targetscan","pictar"),org="Hs",at.least=2,id.out="symbol",verbose= ... [TRUNCATED]

Rd file 'RmiRtc.Rd':
  \usage lines wider than 90 characters:
             readRmiRtc(miRtcObj, correlation = -0.75, exprLev = 1, annotation= NULL, fileName = "miRNA_for_genes")

Rd file 'plotRmiRtc.Rd':
  \usage lines wider than 90 characters:
       plotRmiRtc(miRtcObj,gene_id=NULL,timeunit="Time",legend.x=NULL,legend.y=NULL,svgTips=FALSE,svgname=NULL,height=10,width=15)

These lines will be truncated in the PDF manual.
```

## RMySQL (0.10.11)
Maintainer: Jeroen Ooms <jeroen@berkeley.edu>  
Bug reports: https://github.com/rstats-db/rmysql/issues

0 errors | 0 warnings | 0 notes

## rnaSeqMap (2.32.0)
Maintainer: Michal Okoniewski <michal@fgcz.ethz.ch>

0 errors | 1 warning  | 4 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'rnaSeqMap.tex' failed.
LaTeX errors:
! Package auto-pst-pdf Error: 
    "shell escape" (or "write18") is not enabled:
    auto-pst-pdf will not work!
.
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted


checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘.chr.convert’ ‘.countz’ ‘.munion’ ‘.tunion’ ‘.wytnij’ ‘getBamData’
  ‘newSeqReads’

checking R code for possible problems ... NOTE
.exonCoverage: no visible global function definition for ‘exon.details’
.fillRleList: no visible global function definition for ‘Rle’
.geneCoverage: no visible global function definition for ‘gene.details’
.munion: no visible global function definition for ‘exon.details’
.munion: no visible global function definition for ‘gene.to.exon’
.rsCount: no visible global function definition for ‘gene.details’
.simplePlot: no visible global function definition for ‘plot’
.simplePlot: no visible global function definition for ‘lines’
.tunion: no visible global function definition for ‘exon.details’
... 100 lines ...
Undefined global functions or variables:
  Rle dbGetQuery exon.details gene.details gene.to.exon
  gene.to.transcript ks.test legend lines menu out par phenoData
  phenoData<- plot qqplot read.table rect topo.colors
  transcript.to.exon write.table
Consider adding
  importFrom("grDevices", "topo.colors")
  importFrom("graphics", "legend", "lines", "par", "plot", "rect")
  importFrom("stats", "ks.test", "qqplot")
  importFrom("utils", "menu", "read.table", "write.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'SeqReds.Rd':
  \usage lines wider than 90 characters:
     newSeqReads(chr, start, end, strand, datain=NULL, phenoData=NULL, featureData=NULL, covdesc=NULL)

Rd file 'parseGff3.Rd':
  \usage lines wider than 90 characters:
     parseGff3(filegff, fileg="genes.txt", filet="transcripts.txt", filee="exons.txt", nofiles=FALSE)

Rd file 'regionBasedCoverage.Rd':
  \examples lines wider than 100 characters:
       #runs the Lindell-Aumann algorithm at 100, 90, ... and picks maximal mi-level, where the nucleotide has a region found

These lines will be truncated in the PDF manual.
```

## RObsDat (16.03)
Maintainer: Dominik Reusser <reusser@pik-potsdam.de>

0 errors | 0 warnings | 0 notes

## RODBCDBI (0.1.1)
Maintainer: Nagi Teramo <teramonagi@gmail.com>

0 errors | 0 warnings | 0 notes

## Roleswitch (1.12.0)
Maintainer: Yue Li <yueli@cs.toronto.edu>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘pracma’ ‘reshape’ ‘plotrix’ ‘microRNA’ ‘biomaRt’ ‘Biostrings’
  ‘Biobase’ ‘DBI’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking R code for possible problems ... NOTE
diagnosticPlot: no visible global function definition for ‘par’
diagnosticPlot: no visible global function definition for ‘axis’
diagnosticPlot: no visible global function definition for ‘plot’
getSeedMatrix: no visible global function definition for ‘data’
getTranscriptIDwithLongest3UTR: no visible global function definition
  for ‘aggregate’
roleswitch: no visible global function definition for ‘aggregate’
Undefined global functions or variables:
  aggregate axis data par plot
Consider adding
  importFrom("graphics", "axis", "par", "plot")
  importFrom("stats", "aggregate")
  importFrom("utils", "data")
to your NAMESPACE file.
```

## ROracle (1.3-1)
Maintainer: Rajendra S. Pingte <rajendra.pingte@oracle.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ROracle’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/DBI/revdep/checks/ROracle.Rcheck/00install.out’ for details.
```

## rpostgisLT (0.4.0)
Maintainer: David Bucklin <dbucklin@ufl.edu>  
Bug reports: https://github.com/mablab/rpostgisLT/issues

0 errors | 0 warnings | 0 notes

## rpostgis (1.2.0)
Maintainer: David Bucklin <dbucklin@ufl.edu>  
Bug reports: https://github.com/mablab/rpostgis/issues

0 errors | 0 warnings | 0 notes

## RPostgreSQL (0.4-1)
Maintainer: Tomoaki Nishiyama <tomoakin@staff.kanazawa-u.ac.jp>

0 errors | 0 warnings | 3 notes

```
checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘config.guess’ ‘config.sub’ ‘install-sh’

checking R code for possible problems ... NOTE
postgresqlImportFile: no visible global function definition for
  ‘count.fields’
postgresqlImportFile: no visible global function definition for
  ‘read.table’
Undefined global functions or variables:
  count.fields read.table
Consider adding
  importFrom("utils", "count.fields", "read.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'postgresqlBuildTableDefinition.Rd':
  \usage lines wider than 90 characters:
     postgresqlBuildTableDefinition(dbObj, name, obj, field.types = NULL, row.names = TRUE, ...)

Rd file 'postgresqlSupport.Rd':
  \usage lines wider than 90 characters:
        postgresqlNewConnection(drv, user, password, host, dbname, port, tty, options, forceISOdate = TRUE)

These lines will be truncated in the PDF manual.
```

## RPresto (1.2.1)
Maintainer: Onur Ismail Filiz <onur@fb.com>  
Bug reports: https://github.com/prestodb/RPresto/issues

0 errors | 0 warnings | 0 notes

## RQDA (0.2-8)
Maintainer: HUANG Ronggui <ronggui.huang@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking whether package ‘RQDA’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/DBI/revdep/checks/RQDA.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘rjpod’
```

## RRedshiftSQL (0.1.2)
Maintainer: Michael Treadwell <michael.treadwell@interworks.com>

0 errors | 0 warnings | 0 notes

## RSQLite (1.1-2)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/rstats-db/RSQLite/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 1515 SKIPPED: 13 FAILED: 99
  1. Failure: DBItest[RSQLite]: Driver: data_type_driver (@spec-driver-data-type.R#13) 
  2. Failure: DBItest[RSQLite]: Connection: data_type_connection (@spec-connection-data-type.R#5) 
  3. Failure: DBItest[RSQLite]: Result: fetch_n_bad (@spec-result-fetch.R#73) 
  4. Failure: DBItest[RSQLite]: Result: fetch_n_bad (@spec-result-fetch.R#73) 
  5. Failure: DBItest[RSQLite]: Result: fetch_n_bad (@spec-result-fetch.R#73) 
  6. Failure: DBItest[RSQLite]: Result: fetch_n_good_after_bad (@spec-result-fetch.R#90) 
  7. Failure: DBItest[RSQLite]: Result: fetch_n_good_after_bad (@spec-result-fetch.R#90) 
  8. Failure: DBItest[RSQLite]: Result: fetch_no_return_value (@spec-result-fetch.R#107) 
  9. Failure: DBItest[RSQLite]: Result: fetch_n_more_rows (@spec-result-fetch.R#197) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  7.8Mb
  sub-directories of 1Mb or more:
    libs   7.2Mb
```

## rTRM (1.12.0)
Maintainer: Diego Diez <diego10ruiz@gmail.com>  
Bug reports: https://github.com/ddiez/rTRM/issues

0 errors | 0 warnings | 1 note 

```
checking Rd line widths ... NOTE
Rd file 'findTRM.Rd':
  \usage lines wider than 90 characters:
     findTRM(g, target, query, method = "nsa", max.bridge = 1, extended = FALSE, strict = FALSE, type = "igraph")

Rd file 'plotGraph.Rd':
  \usage lines wider than 90 characters:
     plotGraph(g, layout = layout.fruchterman.reingold, mar = .5, vertex.pch = 21, vertex.cex, vertex.col, vertex.bg, vertex.lwd, edge.col,  ... [TRUNCATED]

Rd file 'plotTRM.Rd':
  \usage lines wider than 90 characters:
     plotTRM(g, layout = layout.fruchterman.reingold, mar = .5, vertex.col, vertex.cex, vertex.lwd, edge.col, edge.lwd, edge.lty, label = TR ... [TRUNCATED]

Rd file 'processBiogrid.Rd':
  \usage lines wider than 90 characters:
     processBiogrid(dblist, org = "human", simplify = TRUE, type = "physical", mimic.old = FALSE)

These lines will be truncated in the PDF manual.
```

## SEERaBomb (2016.2)
Maintainer: Tomas Radivoyevitch <radivot@ccf.org>

0 errors | 0 warnings | 0 notes

## sejmRP (1.3.4)
Maintainer: Piotr Smuda <piotrsmuda@gmail.com>  
Bug reports: http://github.com/mi2-warsaw/sejmRP/issues

0 errors | 0 warnings | 0 notes

## seqplots (1.12.0)
Maintainer: Przemyslaw Stempor <ps562@cam.ac.uk>  
Bug reports: http://github.com/przemol/seqplots/issues

1 error  | 0 warnings | 3 notes

```
checking tests ... ERROR
  Running ‘test-all.R’ [21s/21s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
         xlab, ylab, autoscale, zmin, zmax, xlim, ln.v, s, indi, o_min, o_max, colvec, 
         clspace, pointsize, embed, ggplot, raster, ...)
  7: plotHeatmap(plotset$data, main, labels, legend, keepratio, plotScale, sortrows, clusters, 
         clstmethod, include, ssomt1, ssomt2, cex.main, cex.lab, cex.axis, cex.legend, 
         xlab, ylab, autoscale, zmin, zmax, xlim, ln.v, s, indi, o_min, o_max, colvec, 
         clspace, pointsize, embed, ggplot, raster, ...)
  8: supersom(Hlist, grid = class::somgrid(xdim = ssomt1, ydim = ssomt2, "hexagonal"), 
         rlen = 100, toroidal = TRUE)
  
  testthat results ================================================================
  OK: 63 SKIPPED: 0 FAILED: 1
  1. Error: Test getPlotSetArray function and plotting interfaces (@test1.R#53) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  9.1Mb
  sub-directories of 1Mb or more:
    R          1.2Mb
    doc        2.4Mb
    seqplots   4.9Mb

checking foreign function calls ... NOTE
Foreign function call to a different package:
  .Call("BWGFile_summary", ..., PACKAGE = "rtracklayer")
See chapter ‘System and foreign language interfaces’ in the ‘Writing R
Extensions’ manual.

checking R code for possible problems ... NOTE
getPlotSetArray : <anonymous>: no visible global function definition
  for ‘qt’
getSF : <anonymous>: no visible global function definition for ‘approx’
ggHeatmapPlotWrapper: no visible global function definition for
  ‘colorRampPalette’
ggHeatmapPlotWrapper: no visible binding for global variable ‘Var2’
ggHeatmapPlotWrapper: no visible binding for global variable ‘Var1’
ggHeatmapPlotWrapper: no visible binding for global variable ‘value’
heatmapPlotWrapper: no visible global function definition for
... 42 lines ...
  capture.output colorRampPalette cutree dist hclust image kmeans
  layout lines mtext par plot.new qt rainbow rect rgb title value
Consider adding
  importFrom("grDevices", "adjustcolor", "colorRampPalette", "rainbow",
             "rgb")
  importFrom("graphics", "abline", "axis", "box", "image", "layout",
             "lines", "mtext", "par", "plot.new", "rect", "title")
  importFrom("stats", "approx", "as.dendrogram", "cutree", "dist",
             "hclust", "kmeans", "qt")
  importFrom("utils", "capture.output")
to your NAMESPACE file.
```

## sf (0.4-1)
Maintainer: Edzer Pebesma <edzer.pebesma@uni-muenster.de>  
Bug reports: https://github.com/edzer/sfr/issues/

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 10.2Mb
  sub-directories of 1Mb or more:
    doc    4.0Mb
    libs   4.2Mb
```

## smnet (2.1.1)
Maintainer: Alastair Rushworth <alastairmrushworth@gmail.com>

0 errors | 0 warnings | 0 notes

## snplist (0.17)
Maintainer: Alexander Sibley <alexander.sibley@dm.duke.edu>

0 errors | 0 warnings | 0 notes

## sparklyr (0.5.3)
Maintainer: Javier Luraschi <javier@rstudio.com>  
Bug reports: https://github.com/rstudio/sparklyr/issues

0 errors | 0 warnings | 0 notes

## sparkwarc (0.1.1)
Maintainer: Javier Luraschi <javier@rstudio.com>  
Bug reports: https://github.com/javierluraschi/sparkwarc

0 errors | 0 warnings | 0 notes

## specL (1.8.0)
Maintainer: Christian Panse <cp@fgcz.ethz.ch>, Witold E. Wolski <wewolski@gmail.com>  
Bug reports: https://github.com/fgcz/specL/issues

0 errors | 1 warning  | 3 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-25 (cdsw.Rmd) 
Error: processing vignette 'cdsw.Rmd' failed with diagnostics:
could not find function "doc_date"
Execution halted


checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  merge.specLSet plot.psm plot.psmSet summary.psmSet
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
.onAttach: no visible global function definition for ‘packageVersion’
plot,specLSet: no visible global function definition for ‘draw.circle’
summary,specLSet : <anonymous>: no visible binding for global variable
  ‘iRTpeptides’
Undefined global functions or variables:
  draw.circle iRTpeptides packageVersion
Consider adding
  importFrom("utils", "packageVersion")
to your NAMESPACE file.

checking Rd files ... NOTE
prepare_Rd: ms1.p2069.Rd:28-32: Dropping empty section \references
prepare_Rd: ms1.p2069.Rd:23-26: Dropping empty section \examples
```

## sqldf (0.4-10)
Maintainer: G. Grothendieck <ggrothendieck@gmail.com>  
Bug reports: http://groups.google.com/group/sqldf

0 errors | 1 warning  | 2 notes

```
checking whether package ‘sqldf’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/DBI/revdep/checks/sqldf.Rcheck/00install.out’ for details.

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘tcltk’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
read.csv.sql: no visible global function definition for ‘download.file’
sqldf: no visible global function definition for ‘modifyList’
sqldf: no visible global function definition for ‘head’
Undefined global functions or variables:
  download.file head modifyList
Consider adding
  importFrom("utils", "download.file", "head", "modifyList")
to your NAMESPACE file.
```

## sqliter (0.1.0)
Maintainer: Wilson Freitas <wilson.freitas@gmail.com>

0 errors | 0 warnings | 0 notes

## sqlutils (1.2)
Maintainer: Jason Bryer <jason@bryer.org>  
Bug reports: https://github.com/jbryer/sqlutils/issues

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘RJDBC’ ‘RMySQL’ ‘RODBC’ ‘RPostgreSQL’ ‘RSQLite’ ‘tcltk’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
cacheQuery: no visible global function definition for ‘read.csv’
cacheQuery: no visible global function definition for ‘write.csv’
Undefined global functions or variables:
  read.csv write.csv
Consider adding
  importFrom("utils", "read.csv", "write.csv")
to your NAMESPACE file.
```

## storr (1.0.1)
Maintainer: Rich FitzJohn <rich.fitzjohn@gmail.com>

0 errors | 0 warnings | 0 notes

## stream (1.2-4)
Maintainer: Michael Hahsler <mhahsler@lyle.smu.edu>  
Bug reports: https://github.com/mhahsler/stream/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.2Mb
  sub-directories of 1Mb or more:
    doc    1.6Mb
    libs   4.1Mb
```

## survey (3.31-5)
Maintainer: "Thomas Lumley" <t.lumley@auckland.ac.nz>

0 errors | 0 warnings | 0 notes

## tcpl (1.2.2)
Maintainer: Dayne L Filer <dayne.filer@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  9.9Mb
  sub-directories of 1Mb or more:
    sql   8.7Mb
```

## tigreBrowserWriter (0.1.4)
Maintainer: Antti Honkela <antti.honkela@helsinki.fi>  
Bug reports: https://github.com/PROBIC/tigreBrowserWriter/issues

0 errors | 0 warnings | 0 notes

## tigre (1.28.1)
Maintainer: Antti Honkela <antti.honkela@hiit.fi>  
Bug reports: https://github.com/ahonkela/tigre/issues

0 errors | 0 warnings | 2 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘demos’

checking R code for possible problems ... NOTE
GPPlot: no visible global function definition for ‘polygon’
export.scores: no visible global function definition for ‘png’
export.scores: no visible global function definition for ‘dev.off’
gpdisimLogLikeGradients: no visible global function definition for
  ‘tail’
[,scoreList: no visible global function definition for ‘slotNames’
c,scoreList: no visible global function definition for ‘slotNames’
Undefined global functions or variables:
  dev.off png polygon slotNames tail
Consider adding
  importFrom("grDevices", "dev.off", "png")
  importFrom("graphics", "polygon")
  importFrom("methods", "slotNames")
  importFrom("utils", "tail")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## timeseriesdb (0.2.1)
Maintainer: 'Matthias Bannert' <bannert@kof.ethz.ch>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
createHstore.ts: no visible global function definition for ‘time’
exploreDb : <anonymous>: no visible global function definition for
  ‘plot’
exportTsList : <anonymous>: no visible global function definition for
  ‘time’
exportTsList: no visible global function definition for ‘write.csv2’
importTsList: no visible global function definition for ‘read.csv2’
importTsList : <anonymous>: no visible global function definition for
  ‘as.ts’
... 15 lines ...
  ‘frequency’
zooLikeDateConvert: no visible global function definition for ‘cycle’
Undefined global functions or variables:
  as.ts cycle frequency is.ts legend par plot read.csv2 time ts
  write.csv2
Consider adding
  importFrom("graphics", "legend", "par", "plot")
  importFrom("stats", "as.ts", "cycle", "frequency", "is.ts", "time",
             "ts")
  importFrom("utils", "read.csv2", "write.csv2")
to your NAMESPACE file.
```

## topGO (2.26.0)
Maintainer: Adrian Alexa <adrian.alexa@gmail.com>

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘graph’ ‘Biobase’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘Rgraphviz’ ‘multtest’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Missing object imported by a ':::' call: ‘globaltest:::globaltest’

checking R code for possible problems ... NOTE
GOplot: no visible global function definition for ‘getDefaultAttrs’
GOplot: no visible global function definition for ‘agopen’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getNodeCenter’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘pieGlyph’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getX’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
... 35 lines ...
  getDefaultAttrs getNodeCenter getNodeLW getNodeXY getX getY
  mt.rawp2adjp mt.teststat name pieGlyph scoreInNode toDot txtLabel

Found the following assignments to the global environment:
File ‘topGO/R/topGOtests.R’:
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec, 
    maxSample))[goSize]), .GlobalEnv)
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec))[goSize]), 
    .GlobalEnv)

checking Rd line widths ... NOTE
Rd file 'diagnosticMethods.Rd':
  \examples lines wider than 100 characters:
     sig.tab <- GenTable(GOdata, Fis = resultFisher, KS = resultKS, orderBy = "KS", ranksOf = "Fis", topNodes = 20)

Rd file 'getPvalues.Rd':
  \usage lines wider than 90 characters:
        getPvalues(edata, classlabel, test = "t", alternative = c("greater", "two.sided", "less")[1],
        genesID = NULL, correction = c("none", "Bonferroni", "Holm", "Hochberg", "SidakSS", "SidakSD",

Rd file 'printGraph-methods.Rd':
  \usage lines wider than 90 characters:
                    useFullNames = TRUE, oldSigNodes = NULL, useInfo = c("none", "pval", "counts", "def", "np", "all")[1],
  \examples lines wider than 100 characters:
     printGraph(GOdata, resultFisher, firstSigNodes = 5, fn.prefix = "sampleFile", useInfo = "all", pdfSW = TRUE)

Rd file 'topGOdata-class.Rd':
  \examples lines wider than 100 characters:
                   description = "GO analysis of ALL data: Differential Expression between B-cell and T-cell",

These lines will be truncated in the PDF manual.
```

## toxboot (0.1.1)
Maintainer: Eric D. Watt <eric@ericdwatt.com>  
Bug reports: https://github.com/ericwatt/toxboot/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘rmongodb’
```

## TScompare (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
tfDetails: no visible global function definition for ‘start’
tfDetails: no visible global function definition for ‘end’
Undefined global functions or variables:
  end start
Consider adding
  importFrom("stats", "end", "start")
to your NAMESPACE file.
```

## TSdata (2016.8-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
INFO: Contacting web service with query: https://stats.oecd.org/restsdmx/sdmx.ashx/GetData/QNA/CAN+USA+MEX.B1_GE.CARSA.Q?format=compact_v2
Apr 05, 2017 10:54:02 PM it.bancaditalia.oss.sdmx.client.RestSdmxClient runQuery
INFO: Contacting web service with query: http://ec.europa.eu/eurostat/SDMX/diss-web/rest/dataflow/ESTAT/ei_nama_q/latest
Apr 05, 2017 10:54:02 PM it.bancaditalia.oss.sdmx.client.RestSdmxClient runQuery
INFO: Contacting web service with query: http://ec.europa.eu/eurostat/SDMX/diss-web/rest/dataflow/ESTAT/ei_nama_q/latest
Apr 05, 2017 10:54:02 PM it.bancaditalia.oss.sdmx.client.RestSdmxClient runQuery
INFO: Contacting web service with query: http://ec.europa.eu/eurostat/SDMX/diss-web/rest/datastructure/ESTAT/DSD_ei_nama_q/1.0
... 8 lines ...
INFO: The sdmx call returned messages in the footer:
 Message [code=400, severity=Error, url=null, text=[Error caused by the caller due to incorrect or semantically invalid arguments]]
Apr 05, 2017 10:54:03 PM it.bancaditalia.oss.sdmx.client.RestSdmxClient getData
INFO: The sdmx call returned messages in the footer:
 Message [code=400, severity=Error, url=null, text=[Error caused by the caller due to incorrect or semantically invalid arguments]]

Error: processing vignette 'Guide.Stex' failed with diagnostics:
 chunk 5 
Error in .local(serIDs, con, ...) : 
  ei_nama_q.Q.MIO-EUR.NSA.CP.NA-P72.IT error: it.bancaditalia.oss.sdmx.exceptions.SdmxXmlContentException: The query: ei_nama_q.Q.MIO-EUR.NSA.CP.NA-P72.IT did not match any time series on the provider.
Execution halted
```

## TSdbi (2015.7-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSfame (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
vintageMap: no visible global function definition for ‘read.table’
TSget,character-TSfameLocalConnection: no visible global function
  definition for ‘as.ts’
TSget,character-TSfameLocalConnection: no visible global function
  definition for ‘zoo’
TSget,character-TSfameLocalConnection: no visible global function
  definition for ‘frequency’
TSget,character-TSfameServerConnection: no visible global function
  definition for ‘as.ts’
TSget,character-TSfameServerConnection: no visible global function
  definition for ‘zoo’
TSget,character-TSfameServerConnection: no visible global function
  definition for ‘frequency’
Undefined global functions or variables:
  as.ts frequency read.table zoo
Consider adding
  importFrom("stats", "as.ts", "frequency")
  importFrom("utils", "read.table")
to your NAMESPACE file.
```

## TSmisc (2016.8-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSMySQL (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSodbc (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSPostgreSQL (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSsdmx (2016.8-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSSQLite (2015.4-1)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 0 notes

## TSsql (2015.1-2)
Maintainer: Paul Gilbert <pgilbert.ttv9z@ncf.ca>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
TSgetSQL: no visible global function definition for ‘ts’
TSputSQL: no visible global function definition for ‘is.ts’
TSputSQL: no visible global function definition for ‘frequency’
TSputSQL: no visible global function definition for ‘time’
TSquery: no visible global function definition for ‘ts’
Undefined global functions or variables:
  frequency is.ts time ts
Consider adding
  importFrom("stats", "frequency", "is.ts", "time", "ts")
to your NAMESPACE file.
```

## twitteR (1.1.9)
Maintainer: Jeff Gentry <geoffjentry@gmail.com>

0 errors | 0 warnings | 0 notes

## Uniquorn (1.2.0)
Maintainer: 'Raik Otto' <raik.otto@hu-berlin.de>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘unitTests’
```

## UPMASK (1.1)
Maintainer: Alberto Krone-Martins <algol@sim.ul.pt>

0 errors | 0 warnings | 0 notes

## VariantAnnotation (1.20.3)
Maintainer: Valerie Obenchain <maintainer@bioconductor.org>

0 errors | 1 warning  | 3 notes

```
checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic '[' and siglist 'VCF,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking installed package size ... NOTE
  installed size is  5.1Mb
  sub-directories of 1Mb or more:
    R         2.6Mb
    extdata   1.2Mb

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘snpStats’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported objects imported by ':::' calls:
  ‘BiocGenerics:::replaceSlots’ ‘BiocGenerics:::testPackage’
  ‘Rsamtools:::.RsamtoolsFile’ ‘Rsamtools:::.RsamtoolsFileList’
  ‘Rsamtools:::.io_check_exists’ ‘S4Vectors:::expandByColumnSet’
  ‘S4Vectors:::labeledLine’ ‘S4Vectors:::recycleVector’
  ‘S4Vectors:::selectSome’
  ‘SummarizedExperiment:::.SummarizedExperiment.charbound’
  ‘SummarizedExperiment:::.cbind.DataFrame’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.predictCodingGRangesList: no visible binding for global variable
  ‘GENETIC_CODE’
VRangesForMatching: no visible binding for global variable ‘REF’
VRangesForMatching: no visible binding for global variable ‘ALT’
probabilityToSnpMatrix: no visible global function definition for
  ‘post2g’
import,VcfFile-ANY-ANY: no visible global function definition for
  ‘checkArgFormat’
Undefined global functions or variables:
  ALT GENETIC_CODE REF checkArgFormat post2g
```

## VariantFiltering (1.10.1)
Maintainer: Robert Castelo <robert.castelo@upf.edu>  
Bug reports: https://github.com/rcastelo/VariantFiltering/issues

0 errors | 1 warning  | 4 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘usingVariantFiltering.pdf’ from 415Kb to 153Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking installed package size ... NOTE
  installed size is  7.9Mb
  sub-directories of 1Mb or more:
    R         3.6Mb
    extdata   3.5Mb

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  'S4Vectors:::labeledLine' 'VariantAnnotation:::.checkArgs'
  'VariantAnnotation:::.consolidateHits'
  'VariantAnnotation:::.returnEmpty'
  See the note in ?`:::` about the use of this operator.
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  '.adjustForStrandSense'

checking Rd line widths ... NOTE
Rd file 'MafDb-class.Rd':
  \examples lines wider than 100 characters:
       ## founder mutation in a regulatory element located within the HERC2 gene inhibiting OCA2 expression.

Rd file 'MafDb2-class.Rd':
  \examples lines wider than 100 characters:
       ## founder mutation in a regulatory element located within the HERC2 gene inhibiting OCA2 expression.

Rd file 'VariantFilteringParam-class.Rd':
... 19 lines ...

Rd file 'autosomalRecessiveHeterozygous.Rd':
  \usage lines wider than 90 characters:
                                                                      BPPARAM=bpparam("SerialParam"))

Rd file 'autosomalRecessiveHomozygous.Rd':
  \usage lines wider than 90 characters:
                                                                    use=c("everything", "complete.obs", "all.obs"),
                                                                    BPPARAM=bpparam("SerialParam"))

These lines will be truncated in the PDF manual.

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘phastCons100way.UCSC.hg38’
```

## vmsbase (2.1.3)
Maintainer: Lorenzo D'Andrea <support@vmsbase.org>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘vmsbase’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/DBI/revdep/checks/vmsbase.Rcheck/00install.out’ for details.
```

## WhopGenome (0.9.7)
Maintainer: Ulrich Wittelsbuerger <ulrich.wittelsbuerger@uni-duesseldorf.de>

0 errors | 0 warnings | 0 notes

